										
								</div>
							</section>

						<!-- Four -->
							<section id="four">
								<div class="container">
									<h3>Software</h3>
						 
				<ul>
					<li><a href="https://github.com/astirn/faithful-heteroscedasticity">Faithful Heteroscedastic Regression with Neural Networks</a></li>
					<li><a href="https://github.com/JiayuSuPKU/Smoother/">Smoother</a>: A Unified Modular Framework to Incorporate Structural Dependency in Spatial Omics Data</li>
					<li><a href="https://github.com/an1lam/deepmr">Deep Mendelian randomization</a>: Investigating the causal knowledge of genomic deep learning models</li>
					<li>Phenome-scale causal network discovery with <a href="https://github.com/brielin/bimmer">bidirectional mediated Mendelian randomization</a></li>
					<li><a href="https://github.com/brielin/wwer">Welch-weighted Egger regression</a> reduces false positives due to correlated pleiotropy in Mendelian randomization bimmer</li>
<li>Annotation-free quantification of RNA splicing using <a href="https://github.com/davidaknowles/leafcutter">LeafCutter</a>. </li>
<li><a href="https://github.com/davidaknowles/eagle">Detecting gene x environment interaction effects using allele-specific expression.</a>. </li>
				</ul>
					
Code from older projects:
				<ul>
<li><a href="https://github.com/davidaknowles/gprn">C#/Infer.NET code for Gaussian Process Regression Networks</a>.</li>
<li><a href="https://github.com/davidaknowles/nsfa">Nonparametric sparse factor analysis</a></li>
<li><a href="https://github.com/davidaknowles/dpvc">Dirichlet Process Variable Clustering</a></li>
				</ul>
								
								</div>
							</section>

							<section id="five">
								<div class="container">
									
										<h3>Reports/Abstracts/Presentations</h3>
										<nav id="nav_more">
										<ul class="feature-icons">
											<li class="fa-group"><a href="#abstracts">Workshop papers/conference abstracts</a></li>
											<li class="fa-paperclip"><a href="#reports">Reports/Theses</a></li>
											<li class="fa-video-camera"><a href="#presentations">Presentations</a></li>
										</ul>
									</nav>
							
						<section id="abstracts">			
						<h4>Workshop papers/conference abstracts</h4> 

						<UL>

						<li><b>David A. Knowles</b>, Stanley Ho, Kien Nguyen, Don Morris, Anthony Magliocco, Anindya Sarkar, Daphne Koller, Sylvia Plevritis, Srinivas Chukka, Michael Barnes (2015). <br>
						<b>Machine Learning-based Prognostication of Breast Cancer Recurrence using Tissue Slide Features. </b> <br>
						<i>Pathology Visions</i> Winner: Best Poster in Image Analysis!<br> </li>

						<li><b>David A. Knowles</b>, Stanley Ho, Kien Nguyen, Don Morris, Anthony Magliocco, Anindya Sarkar, Daphne Koller, Srinivas Chukka, Michael Barnes (2014)<br>
						<b>Machine learning-based prognostication of breast cancer recurrence using tissue slide features from H&E and immunohistochemically stained slides.</b> <br> 
						<i>San Antonio Breast Cancer Symposium</i><br> </li>

						<li> David A. Knowles, Leopold Parts, Daniel Glass and John M. Winn <br>
						<b>Inferring a measure of physiological age from multiple ageing related phenotypes.</b> <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoParGlaWin11.pdf">paper</a> <a href="http://videolectures.net/nipsworkshops2011_knowles_phenotypes/">video</a> <br>
						To appear at the NIPS workshop: <a href="http://agbs.kyb.tuebingen.mpg.de/wikis/bg/NIPSPM11">From Statistical Genetics to Predictive Models in Personalized Medicine (NIPS PM 2011)</a> </li>

						<li> David A. Knowles, Leopold Parts, Daniel Glass and John M. Winn (2010) <br>
						<b>Modeling skin and ageing phenotypes using latent variable models in Infer.NET.</b> <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoParGlaWin10.pdf">paper</a> <a href="knowles2010_predictive_medicine_poster.pdf">poster</a><br>
						Poster presented at: <a href="http://nips.cc/Conferences/2010/Program/event.php?ID=1979">Predictive Models in Personalized Medicine Workshop</a>, NIPS 2010, 6-11 December 2010, Vancouver, BC, Canada. </li>

						<li> Knowles, D. and Holmes, S. (2009) <br>
						<b>Statistical tools for ultra-deep pyrosequencing of fast evolving
						  viruses.</b> <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoHol09.pdf">pdf</a> <a href="http://videolectures.net/nipsworkshops09_knowles_smud/">video</a>
						<a href="http://videolectures.net/site/normal_dl/tag=62979/nipsworkshops09_knowles_smud_01.pdf">slides</a><br>
						Presented at: Computational Biology Workshop, NIPS 2009, 7-12 December 2009, Vancouver, BC, Canada. </li>
						</UL>
</section>			

<section id="reports">			
						<h4>Reports/Theses</h4>

						<UL>

						<li> <b> Bayesian non-parametric models and inference for sparse and hierarchical latent structure (2012) <a href="documents/daknowles_thesis.pdf">pdf</a> </b>  <br>
						PhD Thesis, University of Cambridge<br> 
						Supervisor: Zoubin Ghahramani</li>

						<li> <b> Statistical tools for ulta-deep pyrosequencing of fast evolving viruses (2008) <a href="documents/454report.pdf">pdf</a> </b>  <br>
						MSc Bioinformatics and Systems Biology, Imperial College London, Individual Project <br> 
						Supervisor: Professor Susan Holmes, Stanford University</li>

						<li> <b> SBML-ABC: a package for data simulation, parameter inference and model selection, Group Report (2008) <a href="documents/abcsmc_group.pdf">pdf</a> </b>  <br>
						MSc Bioinformatics and Systems Biology, Imperial College London, Group Project<br> 
						Supervisor: Professor Michael Stumpf</li>
						</UL>
					</section>
						
<section id="presentations">	
						<h4>Presentations</h4>

						<UL>

						<li> <a href="documents/bnp_lecture.pdf">Properties of Bayesian nonparametric models and priors over trees</a>. Guest lecture as part of Matt Hoffman's STAT300 class, summer 2013. </li> 

						<li> <a href="documents/alberto.pdf">Diffusion trees as priors.</a> This was a talk I gave about the Dirichlet diffusion tree and Pitman Yor diffusion tree at Collegio Carlo Alberto. </li> 

						<li> <b>Variational methods for nonparametric Bayesian models</b><br> I gave a brief presentation at Microsoft Research summarising some attempts to use variational inference in nonparametric, particularly Dirichlet Process based, models. The slides are <a href="documents/msrpres.pdf">here</a>. </li> 

						</UL>
						</section>
							</div>
						</section>

					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>&copy; Untitled. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollzer.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>
<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>


<script>
	let slideIndex = 0;
	// get list of photos just using ls photos
	const photo_files = `2022 holiday (and birthday) party.jpg
Climbing_and_Art_2023.jpg
NYGC_Xmas_party_2022_with_TLlab.jpg
ASHG2022_hike_San_Jacinto.jpg
ASHG2022_hike_lunch.jpg
ASHG2022_hike_pinecones.jpg
ASHG2022_hike_selfie.jpg
ASHG2022_with_Raj_lab.jpg
Bowling_2021.jpg
Caveman poetry at the 2022 holiday party.jpg
Lab_hike_in_Beacon_2022.jpg
Summer_2021_crew.jpg`.split('\n'); 
	
	showSlides();
	
	function showSlides() {
		let pic = document.getElementById("pic");
		pic.src = "photos/" + photo_files[slideIndex];
		let cap = document.getElementById("caption");
		cap.textContent = photo_files[slideIndex].replace(/_/g," ").replace(".jpg","").replace(".jpeg","");
		slideIndex++;
		if (slideIndex >= photo_files.length) {slideIndex = 0} 
		setTimeout(showSlides, 2000); // Change image every 2 seconds
	}
	</script>

</body>
</html>
