<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html lang="en">
	<head>
		<title>Knowles lab</title>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
		<link rel="stylesheet" href="assets/css/main.css">
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		
		<style>
			.image.face {
						float: left;
						margin: 0 0.5em 0.5em 0;
						top: 0.25em;
						width: 150px;
					}

.img-photo {
  display: block;
  margin-left: auto;
  margin-right: auto;
  height:90%;
}

.slideshow-container {
  height: 600px;
  position: relative;
  margin: auto;
  display: block;
  margin-left: auto;
  margin-right: auto;
}

.text-caption {
  font-size: 24px;
  width: 100%;
  text-align: center;
}

			</style>
	</head>
	<body>

		<!-- Header -->
			<section id="header">
				<header>
					<span class="image avatar"><img src="images/logo.png" alt=""></span>
					<h1 id="logo"><a href="#">Knowles lab</a></h1>
						<p>Machine learning and genomics</p>
				</header>
				<nav id="nav">
					<ul>
						<li><a href="#one">About</a></li>
						<li><a href="#research">Research</a></li>
						<li><a href="#people">People</a></li>
						<li><a href="#publications">Publications</a></li>
						<li><a href="#software">Software</a></li>
						<li><a href="#reports">Reports/Abstracts/Presentations</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://twitter.com/david_a_knowles" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://github.com/davidaknowles/" class="icon fa-github"><span class="label">Github</span></a></li>
						<li><a href="mailto:david.arthur.knowles@gmail.com" class="icon fa-envelope"><span class="label">Email</span></a></li>
					</ul>
				</footer>
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">

						<!-- One -->
							<section id="one">
								<div class="container">
									<header class="major">
										<h2>Knowles lab</h2>
					<p><a href="http://www.nygenome.org/">New York Genome Center</a> / <a href="https://www.columbia.edu/">Columbia University</a></p>
									</header>
									<p>We develop and apply machine learning, statistical genetics, and functional genomics methods to dissect the causal chain from genetic variation to trait, especially human disease. We're particularly interested in RNA splicing changes as a molecular mediator of genetic risk. We collaborate with diverse research groups at NYGC and beyond collecting large-scale genomics datasets in the context of <a href="https://rajlab.org/">neurodegenerative</a> and <a href="https://labs.icahn.mssm.edu/mullinslab/">neuropsychiatric</a> disease, and developing <a href="http://www.nygenome.org/lab-groups-overview/technology-innovation-lab/">novel genomic technologies</a> including <a href="https://satijalab.org/">single cell methods</a>, <a href="http://sanjanalab.org/home.html">forward genetic screens</a> and <a href="https://tllab.org/">long-read transcriptomics</a>.</p>

									<p>If you're interested in machine-learning-flavored genomics, statisical genetics, functional genomics and/or RNA splicing, please get in <a href="mailto:daknowles@nygenome.org">touch</a>! Postdocs and students are eligible for an appointment at Columbia University.</p>	
									
									<p>David teaches <a href="https://docs.google.com/document/d/1_OF-kEl8Sz69Blj6CjanOIZUTKsGPHUbbiIvLHi20bg/edit?usp=sharing">COMS4762</a>, a computer science class at Columbia on machine learning in genetics (Columbia LionMail account required for access), usually in the fall. He also co-chairs the annual <a href="https://mlcb.org/">Machine Learning in Computational Biology</a> meeting (<a href="https://www.youtube.com/@mlcbconf">Youtube channel!</a>).</p>
										
																		
<div class="slideshow-container">
	<img class="img-photo" id="pic" src="photos/ASHG2022_with_Raj_lab.jpg" alt="slideshow"> 
	<div class="text-caption" id="caption">Caption</div>
  </div>
  <br>

<!--
  <ul class="feature-icons">
	<li class="fa-scissors"><a href="#splicing">RNA splicing</a></li>
	<li class="fa-fighter-jet"><a href="#eagle">Gene-by-environment</a></li>
	<li class="fa-microchip">Digital histology</li>
	<li class="fa-wrench"><a href="#vb">Variational methods</a></li>
	<li class="fa-snowflake-o"><a href="#bnp">Bayesian nonparametrics</a></li>
</ul>
-->


									
									<section id="research">
											<div>
												<h2>Research overview</h2>
<p><b>Quantifying alternative splicing across genotypes, cell-types and disease states.</b> Dr Knowles previously developed <a href="#LeafCutter">LeafCutter</a>, a scalable, flexible approach to quantifying alternative splicing (AS) from bulk  RNA-seq. LeafCutter represents AS via the usage rate of each splice junction relative to a group of neighboring junctions. <a href="#Li2016splicing">We</a> and others have used LeafCutter to show that for typical complex diseases a comparable number of risk variants act through AS vs expression. We are working on tools for <a href="#Isaev2023-ax">single-cell</a>, <a href="#Talukder2025-kx">long-read</a> and <a href="#Su2025-by">spatial</a> AS analysis, as well as <a href="#Schertzer2025-pv">ways to interpret</a> the remarkable isoform complexity of human cells revealed by long-read RNA-seq.</p>

<p><b>QTL mapping in thousands of brain samples.</b> RNA-seq sample sizes for brain have lagged behind other tissues. With the <a href="https://rajlab.org/">Raj lab</a> we have uniformly processed over 10,000 post-mortem bulk brain RNA-seq samples, a resource we call <a href='#Real2025-xl'>BigBrain</a>. BigBrain is the largest brain quantitative trait locus (QTL) meta-analysis to date so the resulting QTLs are providing unprecedented resolution to dissect neurological disease GWAS loci. In a parallel effort, <a href="#Jang2025-vj">SingleBrain</a>, we meta-analysed brain single nuclei RNA-seq from ~1000 individuals to map cell-type & state specific expression QTLs.</p>

<p><b>Deep learning models of gene regulation for variant effect prediction.</b> QTL mapping requires large sample sizes and is only practical for common genetic variants and cell-types/states. By contrast, sequence-to-function deep learning (DL) models trained to predict chromatin marks, expression or splicing, from DNA sequence alone, only need data from a few individuals. Dr Knowles was involved in developing <a href="#Jaganathan2019-kb">SpliceAI</a>, one of the earlier methods using modern DL to do this, specifically for splice sites. The predicted difference between the reference and alternate alleles (delta scores) can be powerful predictors of variant effects, even for previously unseen variants. As part of the <a href="https://adsp.niagads.org/">Alzheimer's Disease Sequencing Project</a> (ADSP) we are training and assessing DL models of chromatin accessibility, histone modifications and splicing in AD relevant cell types. Delta scores for microglial chromatin accessibility are strongly enriched in AD GWAS and <a href="#Lakhani2025-fs">enable improved functional fine-mapping</a>. We developed <a href="#Das2025-uy">new rare variant association models</a> to leverage these delta scores and find new genetic associations with AD. </p>

<p><b>Causal gene regulatory network inference.</b> DL models and cis QTL mapping help link molecular changes to disease but don't model how these effects propagate through the gene regulatory network (GRN). Building on <a href="#Brown2021">our earlier work</a> on robust, scalable Mendelian randomization, we developed <a href="#Brown2025-uc">inverse sparse regression</a>, which deconvolves an estimated total causal effect network into direct and indirect effects using a directed analogue of the graphical lasso. We are applying inspre to trans-eQTLs from BigBrain and neurological GWAS to map how genetic effects propagate through the GRN to impact disease risk. To adapt these ideas to the nonlinear, multiple cell-type setting we have <a href="#Lin2025-gg">developed</a> and <a href="#Zweig2025-cd">theoretically characterized</a> differential equation models of cellular dynamics.</p>

<p><b>Experimental characterization of isoform function.</b> It is currently challenging to characterise the function of putative disease relevant isoforms. To address this, we developed a <a href="#Schertzer2025-lb">strategy targeting unique exon-exon junctions</a> in mature RNA with CRISPR/Cas13d to knockdown specific isoforms and, in collaboration with the <a href="https://www.nygenome.org/science-technology/faculty-labs/sanjana-lab/">Sanjana lab</a>, a DL model, <a href="#Wessels2023-ko">TIGER</a>, to predict guide RNA efficacy for this system. </p>
</div>

<p><b>Foundational machine learning.</b> We also work on foundational ML, most recently on <a href="#Su2024-ou">kernel-based latent subspace analysis</a>, <a href="#Stirn2024-qw">principled priors for variational autoencoders</a>, and <a href="#Stirn2023">heteroskedastic regression</a>.</p>

</section> 
								 <!--
<section id="vb"> <div>
									<h4>Variational methods</h4>
									
									<p>Variational methods offer a computationally efficient alternative to Markov chain Monte Carlo algorithms for inference in Bayesian probabilistic models. My work allows such methods to be more easily applied to a broader range of probabilistic models. I extended variational message passing to "non-conjugate" (intuitively, less tractable) models, and incorporated this method, <a href="#Knowles2011c">Non-conjugate VMP</a> (NCVMP), into the publicly available <a href="http://infernet.azurewebsites.net/">Infer.NET</a> software package. Later, Tim Salimans and I did early work on <a href='#salimans2014using'>using Monte Carlo estimation within variational learning</a>, which is now an active subfield of research under the moniker "Stochastic Variational Inference".</p>
	</div> </section>
								 
							<section id="bnp"> <div>
							<h4>Bayesian nonparametric models</h4>
							<p>Bayesian nonparametric (BNP) models are a category of statistical methods that automatically adapt their complexity to observed data. I have developed BNP methods for <a href="#Knowles2014">hierarchical clustering</a>, <a href="#wilson2011gaussian">heteroskedastic multivariate regression</a>, <a href="#palla2012infinite">network data</a>, <a href="#Palla2012nonparametric">variable clustering</a> and <a href='#Knowles2011nonparametric'>nonparametric sparse factor analysis</a> (NSFA). In particular, NSFA is able to adaptively choose an appropriate number of factors from data. I used this method to delineate gene co-expression modules from microarray data, and other researchers have subsequently used it in diverse applications including image denoising and EEG analysis.</p>
</div> </section> -->
								

								</div>
							</section>

						<!-- Two -->
							<section id="people">
								<div class="container">
									<h2>People</h2>
              <section id="david"> <div>
								<h4>David A. Knowles, Principal Investigator</h4>
							<div><span class="image face"><img src="images/avatar.jpg" alt="Profile pic" width="150"></span></div><p>I was a postdoc at Stanford with <a href="http://med.stanford.edu/plevritis.html">Sylvia Plevritis</a> (Computational Systems Biology/Radiology) and <a href="http://pritchardlab.stanford.edu/">Jonathan Pritchard</a> (Genetics) having previously worked with <a href="http://ai.stanford.edu/~koller/">Daphne Koller</a>. Apart from working on splicing (see above), I developed <a href="#Knowles2017gxe">EAGLE</a> to detect gene-by-environment (GxE) effects on the transcriptomic using allele-specific expression and a novel, efficient linear mixed model, <a href="https://github.com/davidaknowles/suez">suez</a> which we used to ls to understand the <a href="#Knowles2018dox">genetic basis of anthracycline-cardiotoxicity</a> using iPSC-derived cardiomyocytes. I did my PhD with <a href="http://learning.eng.cam.ac.uk/zoubin/">Zoubin Ghahramani</a> in the <a href="http://mlg.eng.cam.ac.uk/">Machine Learning</a> group of the Cambridge University Engineering Department, where I worked on <a href='#Knowles2011nonparametric'>Bayesian non-parametric models</a> and <a href='#salimans2014using'>(stochastic)</a> <a href="#Knowles2011c">variational inference</a>. I was the Roger Needham Scholar at Wolfson College, funded by Microsoft Research. My undergraduate degree comprised two years of Physics before switching to Engineering to complete an MEng with Zoubin. I took the <a href="http://www.bioinformatics.ic.ac.uk/">MSc Bioinformatics and Systems Biology</a> at <a href="http://www3.imperial.ac.uk/">Imperial College</a> in 2007/8. <a href="documents/david-knowles-resume.pdf">CV</a>. </p>
							</div> </section>
						
						<section id="staff"> <div>
						 	<h3>Staff</h3>
							<p></p>
        
        <section id="chirag"> <div>
          <h4>Chirag Lakhani - Staff Scientist</h4>
        <div><span class="image face"><img src="images/chirag.jpg" alt="Chirag" width="150"></span></div><p>Chirag is broadly interested in utilizing methods from machine learning and statistical genetics to elucidate the biological function of genetic variants.  Before joining NYGC he was a postdoctoral fellow in the Department of Biomedical Informatics at Harvard Medical School mentored by Chirag Patel.  Before joining Harvard he worked in industry as a data scientist at Zaloni, Inc and HelloWallet.  He received his PhD in mathematics at North Carolina State University.  His undergraduate degree was also in mathematics from North Carolina State University.  Outside of the lab he likes to spend time with his family either at home or exploring the city.</p>
  </div> </section>

  <section id="saikat"> <div>
	<h4>Saikat Banerjee	- Staff Scientist</h4>
  <div><span class="image face"><img src="images/saikat.jpg" alt="saikat" width="150"></span></div><p>Saikat is passionate about understanding the regulatory mechanism of complex diseases. He specializes in developing statistical (Bayesian) methods and computational algorithms for analyzing large-scale multimodal NGS data. Currently, he focuses on studying the network effects of neuropsychiatric disorders. Before joining NYGC, he worked on developing fast Bayesian algorithms as a postdoctoral fellow in the group of Matthew Stephens at the University of Chicago. Previously, he worked at the Max Planck Institute for Multidisciplinary Sciences, Göttingen, with Johannes Söding, developing Bayesian methods for fine mapping and detecting trans-eQTLs. Saikat earned his Ph.D. in computational biophysics. Beyond his work, he enjoys history, philosophy, board games, fountain pen restoration, camping, hiking, and arthouse films. His personal website is <a href="https://saik.at">here</a>.</p>
</div> </section>

			  <section id="megan"> <div>
				<h4>Megan Schertzer - Visiting Staff Scientist</h4>
			<div><span class="image face"><img src="images/megan.jpg" alt="Megan" width="150"></span></div><p>Megan was a postdoc in the Knowles Lab from 2019 to 2025. She's now a Senior Staff Scientist jointly with our collaborator <a href="https://med.virginia.edu/sheynkman-lab/">Gloria Sheynkman</a>. She is interested in integrating human genomics and molecular biology to better understand the role of non-coding RNA in the brain. She originally completed a BS in biochemistry from Lee University in Tennessee, and obtained her PhD under the mentorship of Mauro Calabrese at UNC-Chapel Hill. There, she led a project to study how cis-acting long non-coding RNAs efficiently target and spread Polycomb Repressive Complexes to compact chromatin and repress nearby gene expression. Outside of the lab, she enjoys hiking, playing volleyball, and eating Mexican food.</p> 
</div> </section> <!-- End Megan -->

							</div> </section> <!-- End staff -->
							


						<section> <div>
						 	<h3>Postdocs</h3>
      
<section id="scott"> <div>
	<h4>Scott Adamson</h4>
			  <div><span class="image face"><img src="images/scott.jpg" alt="scott" width="150"></span></div><p>Scott joined the Knowles and Lappalainen labs in fall of 2021. He earned his BS in Environmental Science as well as his PhD in Biomedical Sciences from University of Connecticut. During his PhD, he worked under the supervision of Brenton Graveley to develop massively parallel reporter assays to identify the impact of genetic variance on pre-mRNA splicing, as well as understand splicing regulation more generally. In the Knowles lab, he is interested in developing new ways to predict and test the connection between genotype, splicing, and phenotype.</p> 
      </div> </section>


	  <section id="aline"> <div>
		<h4>Aline Réal</h4>
				  <div><span class="image face"><img src="images/aline.jpg" alt="aline" width="150"></span></div><p>Aline is interested in understanding the underlying mechanisms of gene expression regulation as a means to discovering and functionally interpreting genetic variants associated with splicing that contribute to human disease and traits. 
					Aline was raised in the Aosta Valley, a small region of the Northern part of Italy. She completed her Master Degree in Molecular Biotechnology at the University of Turin and her PhD in genomic and digital health with Emmanouil Dermitzakis, Jörg Seebach and Ana Viñuela. 
					There, she led two projects: the first using long-read direct-RNA-sequencing to identify specific variants associated with splicing and see their effect on transcript isoform expression and structure. With the second project she wanted to gain a deeper understanding of the regulatory germline contribution to cancer and to identify which target genes are involved using functional in-vitro screening assays. 
					Outside of the lab, she enjoys cooking Italian dishes, skiing and playing violin. 
					</p> 
		  </div> </section>

		  <section id="izzy"> <div>
			<h4>Isabella Grabski</h4>
					  <div><span class="image face"><img src="images/izzy.jpg" alt="izzy" width="150"></span></div><p>Isabella (Izzy) joined as a postdoctoral research associate in 2023, jointly with the Satija lab. She is interested in developing statistical and machine learning approaches to address challenges in single-cell genomics data analysis. She completed her PhD in Biostatistics at Harvard University, where she was advised by Professors Rafael Irizarry and Giovanni Parmigiani. Prior to her PhD, she completed her Bachelor's degree at Princeton University in Chemical and Biological Engineering. Outside of research, she enjoys hiking, reading, caring for her pet rats, and exploring the food scene of NYC.
						</p> 
			  </div> </section>


			  <section id="tatsu"> <div>
				<h4>Tatsuhiko Naito</h4>
						  <div><span class="image face"><img src="images/tatsu.jpg" alt="tatsu" width="150"></span></div><p>Tatsuhiko joined the Knowles Lab and Raj Lab as a postdoctoral research fellow in 2023. His research interests lie in the application of machine learning to human genomes to gain a deeper understanding of the role of genetic variants in the pathogenesis of neurological diseases. He completed an MD and a PhD from the University of Tokyo in Japan and has experience working as a neurologist. During his PhD, he developed an HLA imputation method using deep learning and applied it to identify novel associations of HLA with diseases under the mentorship of Yukinori Okada at Osaka University. Additionally, he actively participated in human genetic research, including genome-wide association studies. Outside of the lab, he enjoys running in Central Park and working out at the gym.
							</p> 
				  </div> </section>

				<section id="alex"> <div>
				<h4>Alex Tokolyi</h4>
							<div><span class="image face"><img src="images/alex.jpg" alt="alex" width="150"></span></div><p>Alex is interested in further understanding of the regulatory networks that connect common variation with transcriptional processes, and their downstream impacts on molecular phenotypes & disease. Alex grew up in Melbourne, Australia, completing undergraduate degrees at Monash University in Computer Science, Microbiology, and Molecular biology. During these studies he had the opportunity through a CSL and state-funded  internship to pursue research at the Australian Regenerative Medicine Institute, creating systems to explore spatial transcriptomic data. Alex then performed research with Kathryn Holt in bacterial genomics, and Michael Inouye in asthma transcriptional networks, before moving to England to complete his PhD at the Sanger Institute in the University of Cambridge with Emma Davenport, assessing the genetic architecture of transcript splicing in healthy patients and those suffering sepsis.</p> 
					</div> </section>

				<section id="aaron"> <div>
				<h4>Aaron Zweig</h4>
							<div><span class="image face"><img src="images/aaron.jpg" alt="aaron" width="150"></span></div><p>Aaron joined the Knowles lab as a postdoctoral research associate in 2024, jointly appointed with the Azizi lab at Columbia.  He is excited to apply his computational background with geometric deep learning for causal inference of gene regulatory networks and interpretable spatial transcriptomics.  Aaron studied mathematics for his bachelor’s degree at Stanford University, followed by a master’s degree at Stanford in Stefano Ermon’s lab researching graph neural networks.  He completed his PhD in Joan Bruna’s lab at NYU, where his research concerned approximation bounds and learnability guarantees of permutation-invariant networks.  Outside of research, he enjoys aerial acrobatics, singing, and jaunty historical nonfiction.</p> 
					</div> </section>
					
      
    </div> </section> <!-- End postdocs -->
							
						<section id="phds"> <div>
               <h3>PhD students</h3>

		<section id="jiayu"> <div>
			<h4>Jiayu Su</h4>
		<div><span class="image face"><img src="images/jiayu.jpg" alt="Jiayu" width="150"></span></div><p>Jiayu Su is a PhD student in Systems Biology at Columbia University co-mentored by Raul Rabadan. He is interested in the functional impact and regulation of alternative splicing in cancer and is always enthusiastic to develop new computational tools for genomic applications. Prior to Columbia, Jiayu earned a BS in Biology and a BS in Mathematics from Peking University in 2020, where he explored statistical methods for single-cell genomics and multi-omics. Outside of the lab, he is a history buff and enjoys museums, landmarks, and stories around the world.</p> 
</div> </section> <!-- End Jiayu -->

<section id="anjali"> <div>
	<h4>Anjali Das</h4>
<div><span class="image face"><img src="images/anjali.jpg" alt="anjali" width="150"></span></div><p>Anjali joined the Knowles lab as a PhD student in Computer Science at Columbia University at the start of 2022. Her research interest lies in building and applying machine learning methods to better understand how genetic factors contribute to disease. Anjali graduated from the University of Chicago in 2020 with a BS in Statistics and a minor in computer science, where she researched the genetic basis of Alzheimer’s disease in the Hutterite population. After graduating, she worked as a data scientist at UChicago’s Research Computing Center. In her spare time, she enjoys knitting and hiking.</p> 
</div> </section> <!-- End anjali -->

<section id="sei"> <div>
	<h4>Sei Chang</h4>
<div><span class="image face"><img src="images/sei.jpg" alt="sei" width="150"></span></div><p>Sei is a CS PhD student and NSF Graduate Research Fellow at Columbia University. He is interested in applying deep learning techniques to analyze transcriptomic dysregulation in genetic diseases. Prior to Columbia, Sei graduated from the University of California, Los Angeles in 2022 with a BS in Computer Science and a Minor in Bioinformatics. At UCLA Computational Medicine, he worked on comprehensive benchmarking of structural variant callers. He previously interned at Illumina, developing computational methods for primary analysis in their NGS data pipeline.</p> 
</div> </section> <!-- End Sei -->

<section id="dan"> <div>
	<h4>Dan Meyer</h4>
<div><span class="image face"><img src="images/dan.jpg" alt="dan" width="150"></span></div><p>Dan is a PhD student in Computer Science at Columbia University and an NSF CSGrad4US fellow. Before starting at Columbia in 2023, he worked as a Computational Associate in Steve McCarroll's lab the Broad Institute for 5 years. At the Broad, he helped to develop computational pipelines, methods and analytical frameworks for linking common human genetic variation to quantitative phenotypes using stem cell villages. He is interested in developing machine learning methods for furthering our understanding of how common genetic and epigenetic variation gives rise to disease. He earned a BS in Computer Science from Tufts University in 2018. Outside of research, Dan is a bassoonist, Linux enthusiast, and proud dog parent.</p> 
</div> </section> <!-- End Dan -->

<section id="argha"> <div>
	<h4>Arghamitra Talukder</h4>
<div><span class="image face"><img src="images/argha.jpg" alt="argha" width="150"></span></div><p>Argha is a Computer Science PhD student and NSF CSGrad4US fellow at Columbia University co-advised by Itsik Pe'er. Her research focuses on applying machine learning and statistical modeling to study splicing regulation and isoforms. She completed her bachelors in Electrical Engineering at Texas A&M University where she explored protein interaction prediction using deep learning models. Following graduation, she worked for two years as a Test and Product Engineer at Texas Instruments in Dallas. In her free time, Argha enjoys watching films in different languages, using the experience to learn new language skills.</p> 
</div> </section> <!-- End Argha -->

			</div> </section> <!-- End PhD -->

						<section id="mscs"> <div>
				<h3>Masters students</h3>

				   <section id="julia"> <div>
					<h4>Julia Lewandowski</h4>
				<div><span class="image face"><img src="images/julia.jpg" alt="julia" width="150"></span></div><p>Julia is an MS student in Computer Science at Columbia University. She earned her BS in Computer Science from Dalhousie University, where she worked on applying gene neighborhood embedding-based methods to identify candidate divergent antimicrobial resistance genes. After graduation, she continued in the same lab as a software developer, contributing to a multi-institution project across Canada focused on developing tools to study the emergence, transmission, and ecology of antimicrobial resistance. Currently, Julia's research focuses on developing interpretable machine learning models to elucidate splicing factor (SF) activity from RNA-seq data, with applications in disentangling SF activity in dysregulated settings such as disease contexts. Outside of the lab, she enjoys making art, reading, baking, and caring for her high-energy aussiedoodle.</p> 
				   </div> </section> <!-- End julia -->
			   
				   <section id="smriti"> <div>
					<h4>Smriti Vaidyanathan</h4>
				<div><span class="image face"><img src="images/smriti.jpg" alt="Smriti" width="150"></span></div><p>Smriti is an MS student in Computer Science at Columbia University. She received her Sc.B. in Computational Biology from Brown University, where she studied sex-specific splicing in Drosophila by analyzing the dynamics of RNA-binding protein condensates. Currently, her research focuses on developing deep generative models to integrate gene expression and alternative splicing signals for improved representation of cell states in single-cell data. In her free time, she enjoys singing, going to the gym, trying new foods in the city, and playing The Sims.</p> 
				   </div> </section> <!-- End smriti -->

				   				   <section id="nick"> <div>
					<h4>Nicholas Keung</h4>
				<div><span class="image face"><img src="images/nick.jpg" alt="Nicholas" width="150"></span></div><p>Nick is an MS student at Columbia University studying Computer Science. He is currently applying machine learning to study RNA splicing regulation and alternative splicing isoforms. He received his BS in biology from Emory University. While there, he studied RNA synthesis in RSV and Ebola virus using cryo-EM. In his spare time, Nick enjoys reading, going to the gym, and running in Central Park.</p> 
				   </div> </section> <!-- End nick -->

			 </div> </section> 

      <br>
			 						
			<section id="undergrads"> <div>
				<h3>Undergraduate students</h3>
				
			<section id="gill"> <div>
				<h4>Gill Bartels-Quansah</h4>
			<div><span class="image face"><img src="images/gill.jpg" alt="Gill" width="150"></span></div><p>Gill is an undergraduate student at Barnard College majoring in Computer Science and minoring in English and Science and Public Policy. She is interested in the underlying algorithmic biases in machine learning and its effects on the field of genomics. She is currently studying the role of RNA binding proteins in Alzheimer's Disease using deep learning models. In addition, Gill is interested in the intersection between racial justice, art, and technology which she pursues through her work with Arts and Resistance Through Education (ARTE) and as a Writing Fellow at Barnard. In her free time, Gill enjoys singing with her a cappella group, crocheting, and exploring the city for artsy bookstores.</p> 
			</div> </section> <!-- End Gill -->

			<section id="nicole"> <div>
			<h4>Nicole Cui</h4>
		<div><span class="image face"><img src="images/nicole.jpg" alt="Nicole" width="150"></span></div><p>Nicole is an undergraduate at Columbia SEAS studying computer science and applied math. She is interested in the application of machine learning to understand alternative splicing behavior. Her research at the Knowles Lab focuses on exitrons and their effect on neurodegenerative disease phenotypes. Outside of the lab, Nicole enjoys drawing, hiking, and learning to play guitar.</p> 
			</div> </section> <!-- End nicole -->
			   
       </div> </section> <!-- End UG -->
	<br>
       <section id="Alumni"> <div>
						<h3>Alumni</h3>

		<h4>Staff Scientists</h4>
		<ul>
			<li><a href="https://www.linkedin.com/in/teresa-js-lin/">Jui-Shan (Teresa) Lin</a></li>
			<li><a href="https://www.linkedin.com/in/claire-harbison-1b8ab213b/">Claire Harbison</a></li>
			<li><a href="https://www.linkedin.com/in/laurapereiraphd/">Laura Pereira</a></li>
		</ul>

		<h4>Postdocs</h4>
		<ul>
			<li><a href="https://www.linkedin.com/in/brielin-brown-73745878">Brielin C Brown</a> (now faculty at UPenn)</li>
		</ul>

		<h4>PhD Students</h4>
		<ul>
			<li><a href="https://www.linkedin.com/in/karin-isaev">Karin Isaev</a></li>
			<li><a href="https://www.linkedin.com/in/andrewstirn">Andrew Stirn</a></li>
		</ul>

		<h4>Masters Students</h4>
		<ul>
			<li><a href="https://www.linkedin.com/in/xingpei-zhang">Xingpei Zhang</a></li>
			<li><a href="https://tingtang2.github.io/">Ting Chen</a></li>
			<li>Zaikang (Zach) Lin (now a PhD student with Fabian Theis)</li>
			<li><a href="https://www.linkedin.com/in/stephen-malina">Stephen Malina</a></li>
			<li><a href="https://uk.linkedin.com/in/nasrine-metic">Nasrine Metic</a></li>
			<li><a href="https://uk.linkedin.com/in/chlo%C3%A9-terwagne">Chloé Terwagne</a></li>
		</ul>

		<h4>Undergraduate Students</h4>
		<ul>
			<li><a href="https://www.linkedin.com/in/stella-park-545026214/">Stella Park</a></li>
			<li><a href="https://www.linkedin.com/in/molly-durawa">Molly Durawa</a></li>
			<li><a href="https://www.linkedin.com/in/sophia-sowinski-b31837218">Sophia Sowinski</a></li>
			<li><a href="https://www.linkedin.com/in/collin-w-056029103/">Collin Wang</a></li>
			<li><a href="https://www.linkedin.com/in/fionnafeerick/">Fionna Feerick</a></li>
			<li><a href="https://www.linkedin.com/in/mariya-semchuk/">Mariya Semchuk</a></li>
			<li><a href="https://www.linkedin.com/in/claire-s-johnson/">Claire Johnson</a></li>
			<li><a href="https://www.linkedin.com/in/peter-halmos-680937124">Peter Halmos</a></li>
			<li><a href="https://www.linkedin.com/in/siddhantsanghi">Siddhant Sanghi</a></li>
			<li><a href="https://www.linkedin.com/in/udai-nagpal-76b304149">Udai Nagpal</a></li>
		</ul>

        </div> </section> <!-- End alumni -->

              <section id="join"> <div>
                <h3>Join us!</h3>
							<p> If you're looking at PhD programmes please bear in mind you will need to apply through Columbia University <a href="https://www.cs.columbia.edu/">Computer Science</a> or <a href="https://systemsbiology.columbia.edu/">Systems Biology</a>.</p>
							</div> </section>

							</div> </section>

						<!-- Three -->
							<section id="publications">
								<div class="container">
									<h3>Publications</h3>
									<ul class="feature-icons">
										<li class="fa-bullhorn"><a href="#working">Under submission</a></li>
										<li class="fa-flask"><a href="#genetics">Genetics</a></li>
										<li class="fa-cog"><a href="#ml">Machine learning/statistics</a></li>
									</ul>
