<ol>
<li><div id="Terrey2025-tc"><em>Terrey M, Krivoshein G, Adamson SI, Arystarkhova E, Anderson L, Szwec J, McKee S, Jones H, Perkins S, Selvam V, Piec P-A, Chhaya D, Dehn A, Zuberi A, Murray SA, Morsci NS, Sweadner KJ, Knowles DA, Tolner EA, van den Maagdenberg AMJM and Lutz CM</em> (2025), <strong>"Alternating hemiplegia of childhood associated mutations in Atp1a3 reveal diverse neurological alterations in mice"</strong>, Neurobiology of disease. </div>
<div>[<a data-toggle="collapse" href="#abs_Terrey2025-tc">Abstract</a>][<a href="#bib_Terrey2025-tc" data-toggle="collapse">BibTeX</a>] [<a href="http://dx.doi.org/10.1016/j.nbd.2025.106954" target="_blank">URL</a>]</div>

<div id="abs_Terrey2025-tc" class="collapse">
	<b>Abstract</b>: Pathogenic variants in the neuronal Na+/K+ ATPase transmembrane ion transporter (ATP1A3) cause a spectrum of neurological disorders including alternating hemiplegia of childhood (AHC). The most common de novo pathogenic variants in AHC are p.D801N (∼40 % of patients) and p.E815K (∼25 % of patients), which lead to early mortality by spontaneous death in mice. Nevertheless, knowledge of the development of clinically relevant neurological phenotypes without the obstacle of premature death, is critical for the identification of pathophysiological mechanisms and ultimately, for the testing of therapeutic strategies in disease models. Here, we used hybrid vigor attempting to mitigate the fragility of AHC mice and then performed behavioral, electrophysiological, biochemical, and molecular testing to comparatively analyze mice that carry either of the two most common AHC patient observed variants in the Atp1a3 gene. Collectively, our data reveal the presence but also the differential impact of the p.D801N and p.E815K variants on disease relevant alterations such as spontaneous and stress-induced paroxysmal episodes, motor function, behavioral and neurophysiological activity, and neuroinflammation. Our alternate AHC mouse models with their phenotypic deficits open novel avenues for the investigation of disease biology and therapeutic testing for ATP1A3 research.
</div>
<div id="bib_Terrey2025-tc" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Terrey2025-tc,
  author = {Terrey, Markus and Krivoshein, Georgii and Adamson, Scott I and Arystarkhova, Elena and Anderson, Laura and Szwec, John and McKee, Shelby and Jones, Holly and Perkins, Sara and Selvam, Vijay and Piec, Pierre-Alexandre and Chhaya, Dweet and Dehn, Ari and Zuberi, Aamir and Murray, Stephen A and Morsci, Natalia S and Sweadner, Kathleen J and Knowles, David A and Tolner, Else A and van den Maagdenberg, Arn M J M and Lutz, Cathleen M},
  title = {Alternating hemiplegia of childhood associated mutations in Atp1a3 reveal diverse neurological alterations in mice},
  journal = {Neurobiology of disease},
  publisher = {Elsevier BV},
  year = {2025},
  url = {http://dx.doi.org/10.1016/j.nbd.2025.106954}
}
</pre>
</div>
</li><li><div id="Fernando2024-pw"><em>Fernando MB, Fan Y, Zhang Y, Tokolyi A, Murphy AN, Kammourh S, Deans PJM, Ghorbani S, Onatzevitch R, Pero A, Padilla C, Williams SE, Flaherty EK, Prytkova IA, Cao L, Knowles DA, Fang G, Slesinger PA and Brennand KJ</em> (2025), <strong>"Phenotypic complexities of rare heterozygous neurexin-1 deletions"</strong>, Nature. </div>
<div>[<a data-toggle="collapse" href="#abs_Fernando2024-pw">Abstract</a>][<a href="#bib_Fernando2024-pw" data-toggle="collapse">BibTeX</a>] [<a href="http://dx.doi.org/10.1038/s41586-025-08864-9" target="_blank">URL</a>]</div>

<div id="abs_Fernando2024-pw" class="collapse">
	<b>Abstract</b>: Given the large number of genes significantly associated with risk for neuropsychiatric disorders, a critical unanswered question is the extent to which diverse mutations-sometimes affecting the same gene-will require tailored therapeutic strategies. Here we consider this in the context of rare neuropsychiatric disorder-associated copy number variants (2p16.3) resulting in heterozygous deletions in NRXN1, which encodes a presynaptic cell-adhesion protein that serves as a critical synaptic organizer in the brain. Complex patterns of NRXN1 alternative splicing are fundamental to establishing diverse neurocircuitry, vary between the cell types of the brain and are differentially affected by unique (non-recurrent) deletions1. We contrast the cell-type-specific effect of patient-specific mutations in NRXN1 using human-induced pluripotent stem cells, finding that perturbations in NRXN1 splicing result in divergent cell-type-specific synaptic outcomes. Through distinct loss-of-function (LOF) and gain-of-function (GOF) mechanisms, NRXN1+/- deletions cause decreased synaptic activity in glutamatergic neurons, yet increased synaptic activity in GABAergic neurons. Reciprocal isogenic manipulations causally demonstrate that aberrant splicing drives these changes in synaptic activity. For NRXN1 deletions, and perhaps more broadly, precision medicine will require stratifying patients based on whether their gene mutations act through LOF or GOF mechanisms, to achieve individualized restoration of NRXN1 isoform repertoires by increasing wild-type and/or ablating mutant isoforms. Given the increasing number of mutations predicted to engender both LOF and GOF mechanisms in brain disorders, our findings add nuance to future considerations of precision medicine.
</div>
<div id="bib_Fernando2024-pw" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Fernando2024-pw,
  author = {Fernando, Michael B and Fan, Yu and Zhang, Yanchun and Tokolyi, Alex and Murphy, Aleta N and Kammourh, Sarah and Deans, P J Michael and Ghorbani, Sadaf and Onatzevitch, Ryan and Pero, Adriana and Padilla, Christopher and Williams, Sarah E and Flaherty, Erin K and Prytkova, Iya A and Cao, Lei and Knowles, David A and Fang, Gang and Slesinger, Paul A and Brennand, Kristen J},
  title = {Phenotypic complexities of rare heterozygous neurexin-1 deletions},
  journal = {Nature},
  year = {2025},
  url = {http://dx.doi.org/10.1038/s41586-025-08864-9}
}
</pre>
</div>
</li><li><div id="Das2025-uy"><em>Das A, Lakhani C, Terwagne C, Lin J-ST, Naito T, Raj T and Knowles DA</em> (2025), <strong>"Leveraging functional annotations to map rare variants associated with Alzheimer's disease with gruyere"</strong>, To appear in the American Journal of Human Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Das2025-uy">Abstract</a>][<a href="#bib_Das2025-uy" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2024.12.06.24318577v2" target="_blank">URL</a>]</div>

<div id="abs_Das2025-uy" class="collapse">
	<b>Abstract</b>: The increasing availability of whole-genome sequencing (WGS) has begun to elucidate the contribution of rare variants (RVs), both coding and non-coding, to complex disease. Multiple RV association tests are available to study the relationship between genotype and phenotype, but most are restricted to per-gene models and do not fully leverage the availability of variant-level functional annotations. We propose Genome-wide Rare Variant EnRichment Evaluation (gruyere), a Bayesian probabilistic model that complements existing methods by learning global, trait-specific weights for functional annotations to improve variant prioritization. We apply gruyere to WGS data from the Alzheimer's Disease (AD) Sequencing Project, consisting of 7,966 cases and 13,412 controls, to identify AD-associated genes and annotations. Growing evidence suggests that disruption of microglial regulation is a key contributor to AD risk, yet existing methods have not had sufficient power to examine rare non-coding effects that incorporate such cell-type specific information. To address this gap, we 1) use predicted enhancer and promoter regions in microglia and other potentially relevant cell types (oligodendrocytes, astrocytes, and neurons) to define per-gene non-coding RV test sets and 2) include cell-type specific variant effect predictions (VEPs) as functional annotations. gruyere identifies 15 significant genetic associations not detected by other RV methods and finds deep learning-based VEPs for splicing, transcription factor binding, and chromatin state are highly predictive of functional non-coding RVs. Our study establishes a novel and robust framework incorporating functional annotations, coding RVs, and cell-type associated non-coding RVs, to perform genome-wide association tests, uncovering AD-relevant genes and annotations.
</div>
<div id="bib_Das2025-uy" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Das2025-uy,
  author = {Das, Anjali and Lakhani, Chirag and Terwagne, Chloé and Lin, Jui-Shan T and Naito, Tatsuhiko and Raj, Towfique and Knowles, David A},
  title = {Leveraging functional annotations to map rare variants associated with Alzheimer's disease with gruyere},
  journal = {To appear in the American Journal of Human Genetics},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2024.12.06.24318577v2}
}
</pre>
</div>
</li><li><div id="Talukder2025-kx"><em>Talukder A, Thavarekere S, Mehlferber M, Sheynkman GM and Knowles DA</em> (2025), <strong>"Multi-sample, multi-platform isoform quantification using empirical Bayes"</strong>, RECOMB-Seq. </div>
<div>[<a data-toggle="collapse" href="#abs_Talukder2025-kx">Abstract</a>][<a href="#bib_Talukder2025-kx" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2025.02.08.637184v2.abstract" target="_blank">URL</a>]</div>

<div id="abs_Talukder2025-kx" class="collapse">
	<b>Abstract</b>: Accurate quantification of RNA isoform abundance is crucial for understanding gene regulation, cellular behavior, and disease mechanisms. While short-read (SR) sequencing provides high-throughput and cost-effective transcript quantification, it suffers from read-to-transcript ambiguity. Long-read (LR) sequencing reduces this ambiguity but faces challenges such as high error rates, biases, and lower throughput. Existing methods rely on either SR or LR data and operate on single or merged samples, failing to leverage the variability across multiple samples and the complementary strengths of both technologies. As a result, they struggle to accurately quantify low-abundance and moderate-expressed isoforms and often require complex models for sample-specific bias correction. To address these limitations, we introduce JOLI, a hierarchical model that leverages multi-sample learning to enhance transcript quantification by jointly integrating SR and LR sequencing data. By incorporating multi-sample learning, JOLI captures shared transcript structures, corrects for systematic biases, and enhances statistical power, particularly for low- and moderate-abundance isoforms. Our model applies an empirical Bayes framework, learning a shared prior across samples to improve inference consistency. By jointly modeling SR and LR data, it integrates the strengths of both technologies, achieving higher accuracy and reproducibility in transcript quantification. Through benchmarking on simulated and real RNA-seq datasets, we show that JOLI consistently outperforms single-sample EM method by improving ranking consistency, proportional agreement, and estimation accuracy while enhancing reproducibility. Specifically, in simulations, JOLI multi-sample improves Spearman correlation by 9.8% for LR and 7.7% for SR data compared to single-sample method, while for real data, the improvements are 2.56% (LR) and 1.28% (SR), respectively. Multi-sample learning further improves the quantification of isoforms with low to moderate expression levels. Furthermore, JOLI performs competitively with state-of-the-art methods, highlighting its robustness in transcript quantification.
</div>
<div id="bib_Talukder2025-kx" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Talukder2025-kx,
  author = {Talukder, Arghamitra and Thavarekere, Shree and Mehlferber, Madison and Sheynkman, Gloria M and Knowles, David A},
  title = {Multi-sample, multi-platform isoform quantification using empirical Bayes},
  journal = {RECOMB-Seq},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2025.02.08.637184v2.abstract}
}
</pre>
</div>
</li><li><div id="Park2024-wm"><em>Park CY, Mani S, Beltran-Velez N, Maurer K, Huang T, Li S, Gohil S, Livak KJ, Knowles DA, Wu CJ and Azizi E</em> (2024), <strong>"A Bayesian framework for inferring dynamic intercellular interactions from time-series single-cell data"</strong>, Genome research. </div>
<div>[<a data-toggle="collapse" href="#abs_Park2024-wm">Abstract</a>][<a href="#bib_Park2024-wm" data-toggle="collapse">BibTeX</a>] [<a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&citation_for_view=N9QGV6YAAAAJ:Ei5r6KrKXVQC" target="_blank">URL</a>]</div>

<div id="abs_Park2024-wm" class="collapse">
	<b>Abstract</b>: Characterizing cell-cell communication and tracking its variability over time are crucial for understanding the coordination of biological processes mediating normal development, disease progression, and responses to perturbations such as therapies. Existing tools fail to capture time-dependent intercellular interactions, and primarily rely on existing databases compiled from limited contexts. We introduce DIISCO, a Bayesian framework designed to characterize the temporal dynamics of cellular interactions using single-cell RNA sequencing data from multiple time points. Our method utilizes structured Gaussian process regression to unveil time-resolved interactions among diverse cell types according to their coevolution and incorporates prior knowledge of receptor-ligand complexes. We show the interpretability of DIISCO in simulated data and new data collected from T cells co-cultured with lymphoma cells, demonstrating its potential to uncover dynamic cell-cell crosstalk.
</div>
<div id="bib_Park2024-wm" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Park2024-wm,
  author = {Park, Cameron Y and Mani, Shouvik and Beltran-Velez, Nicolas and Maurer, Katie and Huang, Teddy and Li, Shuqiang and Gohil, Satyen and Livak, Kenneth J and Knowles, David A and Wu, Catherine J and Azizi, Elham},
  title = {A Bayesian framework for inferring dynamic intercellular interactions from time-series single-cell data},
  journal = {Genome research},
  publisher = {Cold Spring Harbor Laboratory},
  year = {2024},
  url = {https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;citation_for_view=N9QGV6YAAAAJ:Ei5r6KrKXVQC}
}
</pre>
</div>
</li><li><div id="Su2023-jg"><em>Su J, Reynier J-B, Fu X, Zhong G, Jiang J, Escalante RS, Wang Y, Aparicio L, Izar B, Knowles DA and Rabadan R</em> (2023), <strong>"Smoother: a unified and modular framework for incorporating structural dependency in spatial omics data"</strong>, Genome biology. </div>
<div>[<a data-toggle="collapse" href="#abs_Su2023-jg">Abstract</a>][<a href="#bib_Su2023-jg" data-toggle="collapse">BibTeX</a>] [<a href="http://dx.doi.org/10.1186/s13059-023-03138-x" target="_blank">URL</a>]</div>

<div id="abs_Su2023-jg" class="collapse">
	<b>Abstract</b>: Spatial omics technologies can help identify spatially organized biological processes, but existing computational approaches often overlook structural dependencies in the data. Here, we introduce Smoother, a unified framework that integrates positional information into non-spatial models via modular priors and losses. In simulated and real datasets, Smoother enables accurate data imputation, cell-type deconvolution, and dimensionality reduction with remarkable efficiency. In colorectal cancer, Smoother-guided deconvolution reveals plasma cell and fibroblast subtype localizations linked to tumor microenvironment restructuring. Additionally, joint modeling of spatial and single-cell human prostate data with Smoother allows for spatial mapping of reference populations with significantly reduced ambiguity.
</div>
<div id="bib_Su2023-jg" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Su2023-jg,
  author = {Su, Jiayu and Reynier, Jean-Baptiste and Fu, Xi and Zhong, Guojie and Jiang, Jiahao and Escalante, Rydberg Supo and Wang, Yiping and Aparicio, Luis and Izar, Benjamin and Knowles, David A and Rabadan, Raul},
  title = {Smoother: a unified and modular framework for incorporating structural dependency in spatial omics data},
  journal = {Genome biology},
  year = {2023},
  url = {http://dx.doi.org/10.1186/s13059-023-03138-x}
}
</pre>
</div>
</li><li><div id="Isaev2023-ax"><em>Isaev K and Knowles DA</em> (2023), <strong>"Investigating RNA splicing as a source of cellular diversity using a binomial mixture model"</strong>, In Proceedings of Machine Learning Research: MLCB 2023. </div>
<div>[<a data-toggle="collapse" href="#abs_Isaev2023-ax">Abstract</a>][<a href="#bib_Isaev2023-ax" data-toggle="collapse">BibTeX</a>] [<a href="https://proceedings.mlr.press/v240/isaev24a.html" target="_blank">URL</a>]</div>

<div id="abs_Isaev2023-ax" class="collapse">
	<b>Abstract</b>: Alternative splicing (AS) contributes significantly to RNA and protein variability yet its role in defining cellular diversity is not fully understood. While Smart-seq2 offers enhanced coverage across transcripts compared to 10X single cell RNA-sequencing (scRNA-seq), current computational methods often miss the full complexity of AS. Most approaches for single cell based differential splicing analysis focus on simple AS events such as exon skipping, and rely on predefined cell type labels or low-dimensional gene expression representations. This limits their ability to detect more complex AS events and makes them dependent on prior knowledge of cell classifications. Here, we present Leaflet, a splice junction centric approach inspired by Leafcutter, our tool for quantifying RNA splicing variation with bulk RNA-seq. Leaflet is a probabilistic mixture model designed to infer AS-driven cell states without the need for cell type labels. We detail Leaflet’s generative model, inference methodology, and its efficiency in detecting differentially spliced junctions. By applying Leaflet to the Tabula Muris brain cell dataset, we highlight cell-state specific splicing patterns, offering a deeper insight into cellular diversity beyond that captured by gene expression alone.
</div>
<div id="bib_Isaev2023-ax" class="collapse">
<b>BibTeX</b>:
<pre>
@inproceedings{Isaev2023-ax,
  author = {Isaev, Keren and Knowles, David A},
  title = {Investigating RNA splicing as a source of cellular diversity using a binomial mixture model},
  booktitle = {Proceedings of Machine Learning Research: MLCB 2023},
  year = {2023},
  url = {https://proceedings.mlr.press/v240/isaev24a.html}
}
</pre>
</div>
</li><li><div id="Brown2023-uc"><em>Brown BC, Morris JA, Lappalainen T and Knowles DA</em> (2025), <strong>"Large-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits"</strong>, To appear in Nature Communications. </div>
<div>[<a data-toggle="collapse" href="#abs_Brown2023-uc">Abstract</a>][<a href="#bib_Brown2023-uc" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2023.10.13.562293v1" target="_blank">URL</a>]</div>

<div id="abs_Brown2023-uc" class="collapse">
	<b>Abstract</b>: Inference of directed biological networks is an important but notoriously challenging problem. We introduce inverse sparse regression (inspre) , an approach to learning causal networks that leverages large-scale intervention-response data. Applied to 788 genes from the genome-wide perturb-seq dataset, inspre helps elucidate the network architecture of blood traits.
</div>
<div id="bib_Brown2023-uc" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Brown2023-uc,
  author = {Brown, Brielin C and Morris, John A and Lappalainen, Tuuli and Knowles, David A},
  title = {Large-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits},
  journal = {To appear in Nature Communications},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2023.10.13.562293v1}
}
</pre>
</div>
</li><li><div id="Schertzer2023-lb"><em>Schertzer MD, Stirn A, Isaev K, Pereira L, Das A, Harbison C, Park SH, Wessels H-H, Sanjana NE and Knowles DA</em> (2025), <strong>"Cas13d-mediated isoform-specific RNA knockdown with a unified computational and experimental toolbox"</strong>, To appear in Nature Communications. </div>
<div>[<a data-toggle="collapse" href="#abs_Schertzer2023-lb">Abstract</a>][<a href="#bib_Schertzer2023-lb" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2023.09.12.557474v1" target="_blank">URL</a>]</div>

<div id="abs_Schertzer2023-lb" class="collapse">
	<b>Abstract</b>: Alternative splicing is an essential mechanism for diversifying proteins, in which mature RNA isoforms produce proteins with potentially distinct functions. Two major challenges in characterizing the cellular function of isoforms are the lack of experimental methods to specifically and efficiently modulate isoform expression and computational tools for complex experimental design. To address these gaps, we developed and methodically tested a strategy which pairs the RNA-targeting CRISPR/Cas13d system with guide RNAs that span exon-exon junctions in the mature RNA. We performed a high-throughput essentiality screen, quantitative RT-PCR assays, and PacBio long read sequencing to affirm our ability to specifically target and robustly knockdown individual RNA isoforms. In parallel, we provide computational tools for experimental design and screen analysis. Considering all possible splice junctions annotated in GENCODE for multi-isoform genes and our gRNA efficacy predictions, we estimate that our junction-centric strategy can uniquely target up to 89% of human RNA isoforms, including 50,066 protein-coding and 11,415 lncRNA isoforms. Importantly, this specificity spans all splicing and transcriptional events, including exon skipping and inclusion, alternative 5' and 3' splice sites, and alternative starts and ends.
</div>
<div id="bib_Schertzer2023-lb" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Schertzer2023-lb,
  author = {Schertzer, Megan D and Stirn, Andrew and Isaev, Keren and Pereira, Laura and Das, Anjali and Harbison, Claire and Park, Stella H and Wessels, Hans-Hermann and Sanjana, Neville E and Knowles, David A},
  title = {Cas13d-mediated isoform-specific RNA knockdown with a unified computational and experimental toolbox},
  journal = {To appear in Nature Communications},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2023.09.12.557474v1}
}
</pre>
</div>
</li><li><div id="Wessels2023-ko"><em>Wessels* H-H, Stirn* A, M&eacute;ndez-Mancilla A, Kim EJ, Hart SK, Knowles† DA and Sanjana† NE</em> (2023), <strong>"Prediction of on-target and off-target activity of CRISPR-Cas13d guide RNAs using deep learning"</strong>, Nature biotechnology.  *Equal contribution. †Co-corresponding.</div>
<div>[<a data-toggle="collapse" href="#abs_Wessels2023-ko">Abstract</a>][<a href="#bib_Wessels2023-ko" data-toggle="collapse">BibTeX</a>] [<a href="https://www.nature.com/articles/s41587-023-01830-8" target="_blank">URL</a>]</div>

<div id="abs_Wessels2023-ko" class="collapse">
	<b>Abstract</b>: Transcriptome engineering applications in living cells with RNA-targeting CRISPR effectors depend on accurate prediction of on-target activity and off-target avoidance. Here we design and test &nbsp;200,000 RfxCas13d guide RNAs targeting essential genes in human cells with systematically designed mismatches and insertions and deletions (indels). We find that mismatches and indels have a position- and context-dependent impact on Cas13d activity, and mismatches that result in G--U wobble pairings are better tolerated than other single-base mismatches. Using this large-scale dataset, we train a convolutional neural network that we term targeted inhibition of gene expression via gRNA design (TIGER) to predict efficacy from guide sequence and context. TIGER outperforms the existing models at predicting on-target and off-target activity on our dataset and published datasets. We show that TIGER scoring combined with specific mismatches yields the first general framework to modulate transcript expression, enabling the use of RNA-targeting CRISPRs to precisely control gene dosage. A machine learning model predicts on-target and off-target activity of Cas13d in human cells.
</div>
<div id="bib_Wessels2023-ko" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Wessels2023-ko,
  author = {Wessels*, Hans-Hermann and Stirn*, Andrew and M&eacute;ndez-Mancilla, Alejandro and Kim, Eric J and Hart, Sydney K and Knowles†, David A and Sanjana†, Neville E},
  title = {Prediction of on-target and off-target activity of CRISPR-Cas13d guide RNAs using deep learning},
  journal = {Nature biotechnology},
  year = {2023},
  url = {https://www.nature.com/articles/s41587-023-01830-8}
}
</pre>
</div>
</li><li><div id="Cortes-Lopez2023-sk"><em>Cort&eacute;s-L&oacute;pez M, Chamely P, Hawkins AG, Stanley RF, Swett AD, Ganesan S, Mouhieddine TH, Dai X, Kluegel L, Chen C, Batta K, Furer N, Vedula RS, Beaulaurier J, Drong AW, Hickey S, Dusaj N, Mullokandov G, Stasiw AM, Su J, Chalign&eacute; R, Juul S, Harrington E, Knowles DA, Potenski CJ, Wiseman DH, Tanay A, Shlush L, Lindsley RC, Ghobrial IM, Taylor J, Abdel-Wahab O, Gaiti F and Landau DA</em> (2023), <strong>"Single-cell multi-omics defines the cell-type-specific impact of splicing aberrations in human hematopoietic clonal outgrowths"</strong>, Cell Stem Cell. </div>
<div>[<a data-toggle="collapse" href="#abs_Cortes-Lopez2023-sk">Abstract</a>][<a href="#bib_Cortes-Lopez2023-sk" data-toggle="collapse">BibTeX</a>] [<a href="https://www.sciencedirect.com/science/article/pii/S1934590923002576" target="_blank">URL</a>]</div>

<div id="abs_Cortes-Lopez2023-sk" class="collapse">
	<b>Abstract</b>: Summary RNA splicing factors are recurrently mutated in clonal blood disorders, but the impact of dysregulated splicing in hematopoiesis remains unclear. To overcome technical limitations, we integrated genotyping of transcriptomes (GoT) with long-read single-cell transcriptomics and proteogenomics for single-cell profiling of transcriptomes, surface proteins, somatic mutations, and RNA splicing (GoT-Splice). We applied GoT-Splice to hematopoietic progenitors from myelodysplastic syndrome (MDS) patients with mutations in the core splicing factor SF3B1. SF3B1mut cells were enriched in the megakaryocytic-erythroid lineage, with expansion of SF3B1mut erythroid progenitor cells. We uncovered distinct cryptic 3′ splice site usage in different progenitor populations and stage-specific aberrant splicing during erythroid differentiation. Profiling SF3B1-mutated clonal hematopoiesis samples revealed that erythroid bias and cell-type-specific cryptic 3′ splice site usage in SF3B1mut cells precede overt MDS. Collectively, GoT-Splice defines the cell-type-specific impact of somatic mutations on RNA splicing, from early clonal outgrowths to overt neoplasia, directly in human samples.
</div>
<div id="bib_Cortes-Lopez2023-sk" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Cortes-Lopez2023-sk,
  author = {Cort&eacute;s-L&oacute;pez, Mariela and Chamely, Paulina and Hawkins, Allegra G and Stanley, Robert F and Swett, Ariel D and Ganesan, Saravanan and Mouhieddine, Tarek H and Dai, Xiaoguang and Kluegel, Lloyd and Chen, Celine and Batta, Kiran and Furer, Nili and Vedula, Rahul S and Beaulaurier, John and Drong, Alexander W and Hickey, Scott and Dusaj, Neville and Mullokandov, Gavriel and Stasiw, Adam M and Su, Jiayu and Chalign&eacute;, Ronan and Juul, Sissel and Harrington, Eoghan and Knowles, David A and Potenski, Catherine J and Wiseman, Daniel H and Tanay, Amos and Shlush, Liran and Lindsley, Robert C and Ghobrial, Irene M and Taylor, Justin and Abdel-Wahab, Omar and Gaiti, Federico and Landau, Dan A},
  title = {Single-cell multi-omics defines the cell-type-specific impact of splicing aberrations in human hematopoietic clonal outgrowths},
  journal = {Cell Stem Cell},
  year = {2023},
  url = {https://www.sciencedirect.com/science/article/pii/S1934590923002576}
}
</pre>
</div>
</li><li><div id="Brown2023-fm"><em>Brown* BC, Wang C, Kasela S, Aguet F, Nachun DC, Taylor KD, Tracy RP, Durda P, Liu Y, Craig Johnson W, Van Den Berg D, Gupta N, Gabriel S, Smith JD, Gerzsten R, Clish C, Wong Q, Papanicolau G, Blackwell TW, Rotter JI, Rich SS, Graham Barr R, Ardlie KG, Knowles* DA and Lappalainen* T</em> (2023), <strong>"Multiset correlation and factor analysis enables exploration of multi-omics data"</strong>, Cell Genomics.  *Co-corresponding.</div>
<div>[<a data-toggle="collapse" href="#abs_Brown2023-fm">Abstract</a>][<a href="#bib_Brown2023-fm" data-toggle="collapse">BibTeX</a>] [<a href="http://www.cell.com/article/S2666979X23001428/abstract" target="_blank">URL</a>]</div>

<div id="abs_Brown2023-fm" class="collapse">
	<b>Abstract</b>: Multi-omics datasets are becoming more common, necessitating better integration methods to realize their revolutionary potential. Here, we introduce multi-set correlation and factor analysis (MCFA), an unsupervised integration method tailored to the unique challenges of high-dimensional genomics data that enables fast inference of shared and private factors. We used MCFA to integrate methylation markers, protein expression, RNA expression, and metabolite levels in 614 diverse samples from the Trans-Omics for Precision Medicine/Multi-Ethnic Study of Atherosclerosis multi-omics pilot. Samples cluster strongly by ancestry in the shared space, even in the absence of genetic information, while private spaces frequently capture dataset-specific technical variation. Finally, we integrated genetic data by conducting a genome-wide association study (GWAS) of our inferred factors, observing that several factors are enriched for GWAS hits and trans-expression quantitative trait loci. Two of these factors appear to be related to metabolic disease. Our study provides a foundation and framework for further integrative analysis of ever larger multi-modal genomic datasets.
</div>
<div id="bib_Brown2023-fm" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Brown2023-fm,
  author = {Brown*, Brielin C and Wang, Collin and Kasela, Silva and Aguet, Fran&cedil; cois and Nachun, Daniel C and Taylor, Kent D and Tracy, Russell P and Durda, Peter and Liu, Yongmei and Craig Johnson, W and Van Den Berg, David and Gupta, Namrata and Gabriel, Stacy and Smith, Joshua D and Gerzsten, Robert and Clish, Clary and Wong, Quenna and Papanicolau, George and Blackwell, Thomas W and Rotter, Jerome I and Rich, Stephen S and Graham Barr, R and Ardlie, Kristin G and Knowles*, David A and Lappalainen*, Tuuli},
  title = {Multiset correlation and factor analysis enables exploration of multi-omics data},
  journal = {Cell Genomics},
  year = {2023},
  url = {http://www.cell.com/article/S2666979X23001428/abstract}
}
</pre>
</div>
</li><li><div id="Malina2022"><em>Malina S, Cizin D and Knowles DA</em> (2022), <strong>"Deep Mendelian randomization: Investigating the causal knowledge of genomic deep learning models"</strong>, PLOS Computational Biology. </div>
<div>[<a data-toggle="collapse" href="#abs_Malina2022">Abstract</a>][<a href="#bib_Malina2022" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pcbi.1009880" target="_blank">DOI</a>] [<a href="https://doi.org/10.1371/journal.pcbi.1009880" target="_blank">URL</a>]</div>

<div id="abs_Malina2022" class="collapse">
	<b>Abstract</b>: Multi-task deep learning (DL) models can accurately predict diverse genomic marks from sequence, but whether these models learn the causal relationships between genomic marks is unknown. Here, we describe Deep Mendelian Randomization (DeepMR), a method for estimating causal relationships between genomic marks learned by genomic DL models. By combining Mendelian randomization with in silico mutagenesis, DeepMR obtains local (locus specific) and global estimates of (an assumed) linear causal relationship between marks. In a simulation designed to test recovery of pairwise causal relations between transcription factors (TFs), DeepMR gives accurate and unbiased estimates of the true global causal effect, but its coverage decays in the presence of sequence-dependent confounding. We then apply DeepMR to examine the global relationships learned by a state-of-the-art DL model, BPNet, between TFs involved in reprogramming. DeepMR's causal effect estimates validate previously hypothesized relationships between TFs and suggest new relationships for future investigation.
</div>
<div id="bib_Malina2022" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Malina2022,
  author = {Malina, Stephen AND Cizin, Daniel AND Knowles, David A.},
  title = {Deep Mendelian randomization: Investigating the causal knowledge of genomic deep learning models},
  journal = {PLOS Computational Biology},
  publisher = {Public Library of Science},
  year = {2022},
  url = {https://doi.org/10.1371/journal.pcbi.1009880},
  doi = {10.1371/journal.pcbi.1009880}
}
</pre>
</div>
</li><li><div id="Humphrey2022"><em>Humphrey J, Venkatesh S, Hasan R, Herb JT, de Paiva Lopes K, K&uuml;&cedil; c&uuml;kali F, Byrska-Bishop M, Evani US, Narzisi G, Fagegaltier D, NYGC ALS Consortium, Sleegers K, Phatnani H, Knowles DA, Fratta P and Raj T</em> (2022), <strong>"Integrative transcriptomic analysis of the amyotrophic lateral sclerosis spinal cord implicates glial activation and suggests new risk genes"</strong>, Nature Neuroscience. </div>
<div>[<a data-toggle="collapse" href="#abs_Humphrey2022">Abstract</a>][<a href="#bib_Humphrey2022" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2021.08.31.21262682v1" target="_blank">URL</a>]</div>

<div id="abs_Humphrey2022" class="collapse">
	<b>Abstract</b>: Amyotrophic lateral sclerosis (ALS) is a progressively fatal neurodegenerative disease affecting motor neurons in the brain and spinal cord. In this study, we investigated gene expression changes in ALS via RNA sequencing in 380 postmortem samples from cervical, thoracic and lumbar spinal cord segments from 154 individuals with ALS and 49 control individuals. We observed an increase in microglia and astrocyte gene expression, accompanied by a decrease in oligodendrocyte gene expression. By creating a gene co-expression network in the ALS samples, we identified several activated microglia modules that negatively correlate with retrospective disease duration. We mapped molecular quantitative trait loci and found several potential ALS risk loci that may act through gene expression or splicing in the spinal cord and assign putative cell types for FNBP1, ACSL5, SH3RF1 and NFASC. Finally, we outline how common genetic variants associated with splicing of C9orf72 act as proxies for the well-known repeat expansion, and we use the same mechanism to suggest ATXN3 as a putative risk gene.
</div>
<div id="bib_Humphrey2022" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Humphrey2022,
  author = {Humphrey, Jack and Venkatesh, Sanan and Hasan, Rahat and Herb, Jake T and de Paiva Lopes, Katia and K&uuml;&cedil; c&uuml;kali, Fahri and Byrska-Bishop, Marta and Evani, Uday S and Narzisi, Giuseppe and Fagegaltier, Delphine and NYGC ALS Consortium and Sleegers, Kristel and Phatnani, Hemali and Knowles, David A and Fratta, Pietro and Raj, Towfique},
  title = {Integrative transcriptomic analysis of the amyotrophic lateral sclerosis spinal cord implicates glial activation and suggests new risk genes},
  journal = {Nature Neuroscience},
  year = {2022},
  url = {https://www.medrxiv.org/content/10.1101/2021.08.31.21262682v1}
}
</pre>
</div>
</li><li><div id="Brown2021"><em>Brown BC and Knowles DA</em> (2021), <strong>"Welch-weighted Egger regression reduces false positives due to correlated pleiotropy in Mendelian randomization"</strong>, American Journal of Human Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Brown2021">Abstract</a>][<a href="#bib_Brown2021" data-toggle="collapse">BibTeX</a>] [<a href="https://www.cell.com/ajhg/fulltext/S0002-9297(21)00383-9" target="_blank">URL</a>]</div>

<div id="abs_Brown2021" class="collapse">
	<b>Abstract</b>: Modern population-scale biobanks contain simultaneous measurements of many phenotypes, providing unprecedented opportunity to study the relationship between biomarkers and disease. However, inferring causal effects from observational data is notoriously challenging. Mendelian randomization (MR) has recently received increased attention as a class of methods for estimating causal effects using genetic associations. However, standard methods result in pervasive false positives when two traits share a heritable, unobserved common cause. This is the problem of correlated pleiotropy. Here, we introduce a flexible framework for simulating traits with a common genetic confounder that generalizes recently proposed models, as well as a simple approach we call Welch-weighted Egger regression (WWER) for estimating causal effects. We show in comprehensive simulations that our method substantially reduces false positives due to correlated pleiotropy while being fast enough to apply to hundreds of phenotypes. We apply our method first to a subset of the UK Biobank consisting of blood traits and inflammatory disease, and then to a broader set of 411 heritable phenotypes. We detect many effects with strong literature support, as well as numerous behavioral effects that appear to stem from physician advice given to people at high risk for disease. We conclude that WWER is a powerful tool for exploratory data analysis in ever-growing databases of genotypes and phenotypes.
</div>
<div id="bib_Brown2021" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Brown2021,
  author = {Brown, Brielin C and Knowles, David A},
  title = {Welch-weighted Egger regression reduces false positives due to correlated pleiotropy in Mendelian randomization},
  journal = {American Journal of Human Genetics},
  year = {2021},
  url = {https://www.cell.com/ajhg/fulltext/S0002-9297(21)00383-9}
}
</pre>
</div>
</li><li><div id="Hadi2020"><em>Hadi K, Yao X, Behr JM, Deshpande A, Xanthopoulakis C, Rosiene J, Darmofal M, Tian H, DeRose J, Mortensen R, Adney EM, Gajic Z, Eng K, Wala JA, Wrzeszczy&#329;ski KO, Arora K, Shah M, Emde A-K, Felice V, Frank MO, Darnell RB, Ghandi M, Huang F, Maciejowski J, De Lange T, Setton J, Riaz N, Reis-Filho JS, Powell S, Knowles D, Reznik E, Mishra B, Beroukhim R, Zody M, Robine N, Oman KM, Sanchez CA, Kuhner MK, Smith LP, Galipeau PC, Paulson TG, Reid BJ, Li X, Wilkes D, Sboner A, Mosquera JM, Elemento O and Imielinski M</em> (2020), <strong>"Novel patterns of complex structural variation revealed across thousands of cancer genome graphs"</strong>, Cell. </div>
<div>[<a data-toggle="collapse" href="#abs_Hadi2020">Abstract</a>][<a href="#bib_Hadi2020" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.cell.2020.08.006" target="_blank">DOI</a>] [<a href="https://www.cell.com/cell/pdf/S0092-8674(20)30997-1.pdf" target="_blank">URL</a>]</div>

<div id="abs_Hadi2020" class="collapse">
	<b>Abstract</b>: Cancer genomes often harbor hundreds of somatic DNA rearrangement junctions, many of which cannot be easily classified into simple (e.g. deletion, translocation) or complex (e.g. chromothripsis, chromoplexy) structural variant classes. Applying a novel genome graph computational paradigm to analyze the topology of junction copy number (JCN) across 2,833 tumor whole genome sequences (WGS), we introduce three complex rearrangement phenomena: pyrgo, rigma, and tyfonas. Pyrgo are &ldquo;towers&rdquo; of low-JCN duplications associated with early replicating regions and superenhancers, and are enriched in breast and ovarian cancers. Rigma comprise &ldquo;chasms&rdquo; of low-JCN deletions at late-replicating fragile sites in esophageal and other gastrointestinal (GI) adenocarcinomas. Tyfonas are &ldquo;typhoons&rdquo; of high-JCN junctions and fold back inversions that are enriched in acral but not cutaneous melanoma and associated with a previously uncharacterized mutational process of non-APOBEC kataegis. Clustering of tumors according to genome graph-derived features identifies subgroups associated with DNA repair defects and poor prognosis.
</div>
<div id="bib_Hadi2020" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Hadi2020,
  author = {Hadi, Kevin and Yao, Xiaotong and Behr, Julie M. and Deshpande, Aditya and Xanthopoulakis, Charalampos and Rosiene, Joel and Darmofal, Madison and Tian, Huasong and DeRose, Joseph and Mortensen, Rick and Adney, Emily M. and Gajic, Zoran and Eng, Kenneth and Wala, Jeremiah A. and Wrzeszczy&#329;ski, Kazimierz O. and Arora, Kanika and Shah, Minita and Emde, Anne-Katrin and Felice, Vanessa and Frank, Mayu O. and Darnell, Robert B. and Ghandi, Mahmoud and Huang, Franklin and Maciejowski, John and De Lange, Titia and Setton, Jeremy and Riaz, Nadeem and Reis-Filho, Jorge S. and Powell, Simon and Knowles, David and Reznik, Ed and Mishra, Bud and Beroukhim, Rameen and Zody, Michael and Robine, Nicolas and Oman, Kenji M. and Sanchez, Carissa A. and Kuhner, Mary K. and Smith, Lucian P. and Galipeau, Patricia C. and Paulson, Thomas G. and Reid, Brian J. and Li, Xiaohong and Wilkes, David and Sboner, Andrea and Mosquera, Juan Miguel and Elemento, Olivier and Imielinski, Marcin},
  title = {Novel patterns of complex structural variation revealed across thousands of cancer genome graphs},
  journal = {Cell},
  year = {2020},
  url = {https://www.cell.com/cell/pdf/S0092-8674(20)30997-1.pdf},
  doi = {10.1016/j.cell.2020.08.006}
}
</pre>
</div>
</li><li><div id="Gentles2019"><em>Gentles AJ, Hui AB-Y, Feng W, Azizi A, Nair RV, Knowles DA, Yu A, Jeong Y, Bejnood A, Forg&oacute; E, Varma S, Xu Y, Kuong A, Nair VS, West R, van de Rijn M, Hoang CD, Diehn M and Plevritis SK</em> (2020), <strong>"Clinically-relevant cell type cross-talk identified from a human lung tumor microenvironment interactome"</strong>, Genome Biology. </div>
<div>[<a data-toggle="collapse" href="#abs_Gentles2019">Abstract</a>][<a href="#bib_Gentles2019" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/637306" target="_blank">DOI</a>] [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02019-x" target="_blank">URL</a>]</div>

<div id="abs_Gentles2019" class="collapse">
	<b>Abstract</b>: Tumors comprise a complex microenvironment of interacting malignant and stromal cell types. Much of our understanding of the tumor microenvironment comes from in vitro studies isolating the interactions between malignant cells and a single stromal cell type, often along a single pathway. To develop a deeper understanding of the interactions between cells within human lung tumors we performed RNA-seq profiling of flow-sorted malignant cells, endothelial cells, immune cells, fibroblasts, and bulk cells from freshly resected human primary non-small-cell lung tumors. We mapped the cell-specific differential expression of prognostically-associated secreted factors and cell surface genes, and computationally reconstructed cross-talk between these cell types to generate a novel resource we call the Lung Tumor Microenvironment Interactome (LTMI). Using this resource, we identified and validated a prognostically unfavorable influence of Gremlin-1 production by fibroblasts on proliferation of malignant lung adenocarcinoma cells. We also found a prognostically favorable association between infiltration of mast cells and less aggressive tumor cell behavior. These results illustrate the utility of the LTMI as a resource for generating hypotheses concerning tumor-microenvironment interactions that may have prognostic and therapeutic relevance.  Summary RNA-seq profiling of sorted populations from primary lung cancer samples identifies prognostically relevant cross-talk between cell types in the tumor microenvironment.
</div>
<div id="bib_Gentles2019" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Gentles2019,
  author = {Gentles, Andrew J and Hui, Angela Bik-Yu and Feng, Weiguo and Azizi, Armon and Nair, Ramesh V. and Knowles, David A. and Yu, Alice and Jeong, Youngtae and Bejnood, Alborz and Forg&oacute;, Erna and Varma, Sushama and Xu, Yue and Kuong, Amanda and Nair, Viswam S. and West, Rob and van de Rijn, Matt and Hoang, Chuong D. and Diehn, Maximilian and Plevritis, Sylvia K.},
  title = {Clinically-relevant cell type cross-talk identified from a human lung tumor microenvironment interactome},
  journal = {Genome Biology},
  year = {2020},
  url = {https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02019-x},
  doi = {10.1101/637306}
}
</pre>
</div>
</li><li><div id="Jaganathan2019-kb"><em>Jaganathan K, Kyriazopoulou Panagiotopoulou S, McRae JF, Darbandi SF, Knowles D, Li YI, Kosmicki JA, Arbelaez J, Cui W, Schwartz GB, Chow ED, Kanterakis E, Gao H, Kia A, Batzoglou S, Sanders SJ and Farh KK-H</em> (2019), <strong>"Predicting Splicing from Primary Sequence with Deep Learning"</strong>, Cell. </div>
<div>[<a data-toggle="collapse" href="#abs_Jaganathan2019-kb">Abstract</a>][<a href="#bib_Jaganathan2019-kb" data-toggle="collapse">BibTeX</a>]</div>

<div id="abs_Jaganathan2019-kb" class="collapse">
	<b>Abstract</b>: The splicing of pre-mRNAs into mature transcripts is remarkable for its precision, but the mechanisms by which the cellular machinery achieves such specificity are incompletely understood. Here, we describe a deep neural network that accurately predicts splice junctions from an arbitrary pre-mRNA transcript sequence, enabling precise prediction of noncoding genetic variants that cause cryptic splicing. Synonymous and intronic mutations with predicted splice-altering consequence validate at a high rate on RNA-seq and are strongly deleterious in the human population. De novo mutations with predicted splice-altering consequence are significantly enriched in patients with autism and intellectual disability compared to healthy controls and validate against RNA-seq in 21 out of 28 of these patients. We estimate that 9%-11% of pathogenic mutations in patients with rare genetic disorders are caused by this previously underappreciated class of disease variation.
</div>
<div id="bib_Jaganathan2019-kb" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Jaganathan2019-kb,
  author = {Jaganathan, Kishore and Kyriazopoulou Panagiotopoulou, Sofia and McRae, Jeremy F and Darbandi, Siavash Fazel and Knowles, David and Li, Yang I and Kosmicki, Jack A and Arbelaez, Juan and Cui, Wenwu and Schwartz, Grace B and Chow, Eric D and Kanterakis, Efstathios and Gao, Hong and Kia, Amirali and Batzoglou, Serafim and Sanders, Stephan J and Farh, Kyle Kai-How},
  title = {Predicting Splicing from Primary Sequence with Deep Learning},
  journal = {Cell},
  year = {2019}
}
</pre>
</div>
</li><li><div id="Wainberg2019-lb"><em>Wainberg M, Sinnott-Armstrong N, Mancuso N, Barbeira AN, Knowles DA, Golan D, Ermel R, Ruusalepp A, Quertermous T, Hao K, Bj&ouml;rkegren JLM, Im HK, Pasaniuc B, Rivas MA and Kundaje A</em> (2019), <strong>"Opportunities and challenges for transcriptome-wide association studies"</strong>, Nature Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Wainberg2019-lb">Abstract</a>][<a href="#bib_Wainberg2019-lb" data-toggle="collapse">BibTeX</a>]</div>

<div id="abs_Wainberg2019-lb" class="collapse">
	<b>Abstract</b>: Transcriptome-wide association studies (TWAS) integrate genome-wide association studies (GWAS) and gene expression datasets to identify gene-trait associations. In this Perspective, we explore properties of TWAS as a potential approach to prioritize causal genes at GWAS loci, by using simulations and case studies of literature-curated candidate causal genes for schizophrenia, low-density-lipoprotein cholesterol and Crohn's disease. We explore risk loci where TWAS accurately prioritizes the likely causal gene as well as loci where TWAS prioritizes multiple genes, some likely to be non-causal, owing to sharing of expression quantitative trait loci (eQTL). TWAS is especially prone to spurious prioritization with expression data from non-trait-related tissues or cell types, owing to substantial cross-cell-type variation in expression levels and eQTL strengths. Nonetheless, TWAS prioritizes candidate causal genes more accurately than simple baselines. We suggest best practices for causal-gene prioritization with TWAS and discuss future opportunities for improvement. Our results showcase the strengths and limitations of using eQTL datasets to determine causal genes at GWAS loci.
</div>
<div id="bib_Wainberg2019-lb" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Wainberg2019-lb,
  author = {Wainberg, Michael and Sinnott-Armstrong, Nasa and Mancuso, Nicholas and Barbeira, Alvaro N and Knowles, David A and Golan, David and Ermel, Raili and Ruusalepp, Arno and Quertermous, Thomas and Hao, Ke and Bj&ouml;rkegren, Johan L M and Im, Hae Kyung and Pasaniuc, Bogdan and Rivas, Manuel A and Kundaje, Anshul},
  title = {Opportunities and challenges for transcriptome-wide association studies},
  journal = {Nature Genetics},
  year = {2019}
}
</pre>
</div>
</li><li><div id="Calderon2018immune"><em>Calderon D, Nguyen ML, Mezger A, Kathiria A, Nguyen V, Lescano N, Wu B, Trombetta J, Ribado JV, Knowles DA, Gao Z, Parent AV, Burt TD, Anderson MS, Criswell LA, Greenleaf WJ, Marson A and Pritchard JK</em> (2019), <strong>"Landscape of stimulation-responsive chromatin across diverse human immune cells"</strong>, Nature Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Calderon2018immune">Abstract</a>][<a href="#bib_Calderon2018immune" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/early/2018/09/05/409722" target="_blank">URL</a>]</div>

<div id="abs_Calderon2018immune" class="collapse">
	<b>Abstract</b>: The immune system is controlled by a balanced interplay among specialized cell types transitioning between resting and stimulated states. Despite its importance, the regulatory landscape of this system has not yet been fully characterized. To address this gap, we collected ATAC-seq and RNA-seq data under resting and stimulated conditions for 25 immune cell types from peripheral blood of four healthy individuals, and seven cell types from three fetal thymus samples. We found that stimulation caused widespread chromatin remodeling, including a large class of response elements shared between stimulated B and T cells. Furthermore, several autoimmune traits showed significant heritability in stimulation-responsive elements from distinct cell types, highlighting the critical importance of these cell states in autoimmunity. Use of allele-specific read-mapping identified thousands of variants that alter chromatin accessibility in particular conditions. Notably, variants associated with changes in stimulation-specific chromatin accessibility were not enriched for associations with gene expression regulation in whole blood -- a tissue commonly used in eQTL studies. Thus, large-scale maps of variants associated with gene regulation lack a condition important for understanding autoimmunity. As a proof-of-principle we identified variant rs6927172, which links stimulated T cell-specific chromatin dysregulation in the TNFAIP3 locus to ulcerative colitis and rheumatoid arthritis. Overall, our results provide a broad resource of chromatin landscape dynamics and highlight the need for large-scale characterization of effects of genetic variation in stimulated cells.
</div>
<div id="bib_Calderon2018immune" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Calderon2018immune,
  author = {Calderon, Diego and Nguyen, Michelle L.T. and Mezger, Anja and Kathiria, Arwa and Nguyen, Vinh and Lescano, Ninnia and Wu, Beijing and Trombetta, John and Ribado, Jessica V. and Knowles, David A. and Gao, Ziyue and Parent, Audrey V. and Burt, Trevor D. and Anderson, Mark S. and Criswell, Lindsey A. and Greenleaf, William J. and Marson, Alexander and Pritchard, Jonathan K.},
  title = {Landscape of stimulation-responsive chromatin across diverse human immune cells},
  journal = {Nature Genetics},
  year = {2019},
  url = {https://www.biorxiv.org/content/early/2018/09/05/409722}
}
</pre>
</div>
</li><li><div id="Knowles2018dox"><em>Knowles* DA, Burrows* CK, Blischak JD, Patterson KM, Serie DJ, Norton N, Ober C, Pritchard JK and Gilad Y</em> (2018), <strong>"Determining the genetic basis of anthracycline-cardiotoxicity by molecular response QTL mapping in induced cardiomyocytes"</strong>, eLife.  *These authors contributed equally to this work.</div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2018dox">Abstract</a>][<a href="#bib_Knowles2018dox" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.7554/eLife.33480" target="_blank">DOI</a>] [<a href="https://elifesciences.org/articles/33480" target="_blank">URL</a>]</div>

<div id="abs_Knowles2018dox" class="collapse">
	<b>Abstract</b>: Anthracycline-induced cardiotoxicity (ACT) is a key limiting factor in setting optimal chemotherapy regimes for cancer patients, with almost half of patients expected to ultimately develop congestive heart failure given high drug doses. However, the genetic basis of sensitivity to anthracyclines such as doxorubicin remains unclear. To begin addressing this, we created a panel of iPSC-derived cardiomyocytes from 45 individuals and performed RNA-seq after 24h exposure to varying levels of doxorubicin. The transcriptomic response to doxorubicin is substantial, with the majority of genes being differentially expressed across treatments of different concentrations and over 6000 genes showing evidence of differential splicing. Overall, our observations indicate that splicing fidelity decreases in the presence of doxorubicin. We detect 376 response-expression QTLs and 42 response-splicing QTLs, i.e. genetic variants that modulate the individual transcriptomic response to doxorubicin in terms of expression and splicing changes respectively. We show that inter-individual variation in transcriptional response is predictive of cell damage measured <em>in vitro</em> using a cardiac troponin assay, which in turn is shown to be associated with <em>in vivo</em> ACT risk. Finally, the molecular QTLs we detected are enriched in lower ACT GWAS p-values, further supporting the <em>in vivo</em> relevance of our map of genetic regulation of cellular response to anthracyclines.
</div>
<div id="bib_Knowles2018dox" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2018dox,
  author = {Knowles*, David A and Burrows*, Courtney K and Blischak, John D and Patterson, Kristen M and Serie, Daniel J. and Norton, Nadine and Ober, Carole and Pritchard, Jonathan K and Gilad, Yoav},
  title = {Determining the genetic basis of anthracycline-cardiotoxicity by molecular response QTL mapping in induced cardiomyocytes},
  journal = {eLife},
  year = {2018},
  url = {https://elifesciences.org/articles/33480},
  doi = {10.7554/eLife.33480}
}
</pre>
</div>
</li><li><div id="Leland_Taylor2018-lb"><em>Leland Taylor D, Knowles DA, Scott LJ, Ramirez AH, Casale FP, Wolford BN, Guan L, Varshney A, Albanus RD, Parker SCJ, Narisu N, Chines PS, Erdos MR, Welch RP, Kinnunen L, Saramies J, Sundvall J, Lakka TA, Laakso M, Tuomilehto J, Koistinen HA, Stegle O, Boehnke M, Birney E and Collins FS</em> (2018), <strong>"Interactions between genetic variation and cellular environment in skeletal muscle gene expression"</strong>, PLoS One. </div>
<div>[<a data-toggle="collapse" href="#abs_Leland_Taylor2018-lb">Abstract</a>][<a href="#bib_Leland_Taylor2018-lb" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pone.0195788" target="_blank">DOI</a>]</div>

<div id="abs_Leland_Taylor2018-lb" class="collapse">
	<b>Abstract</b>: From whole organisms to individual cells, responses to environmental conditions are influenced by genetic makeup, where the effect of genetic variation on a trait depends on the environmental context. RNA-sequencing quantifies gene expression as a molecular trait, and is capable of capturing both genetic and environmental effects. In this study, we explore opportunities of using allele-specific expression (ASE) to discover cis-acting genotype-environment interactions (GxE)---genetic effects on gene expression that depend on an environmental condition. Treating 17 common, clinical traits as approximations of the cellular environment of 267 skeletal muscle biopsies, we identify 10 candidate environmental response expression quantitative trait loci (reQTLs) across 6 traits (12 unique gene-environment trait pairs; 10% FDR per trait) including sex, systolic blood pressure, and low-density lipoprotein cholesterol. Although using ASE is in principle a promising approach to detect GxE effects, replication of such signals can be challenging as validation requires harmonization of environmental traits across cohorts and a sufficient sampling of heterozygotes for a transcribed SNP. Comprehensive discovery and replication will require large human transcriptome datasets, or the integration of multiple transcribed SNPs, coupled with standardized clinical phenotyping.
</div>
<div id="bib_Leland_Taylor2018-lb" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Leland_Taylor2018-lb,
  author = {Leland Taylor, D and Knowles, David A and Scott, Laura J and Ramirez, Andrea H and Casale, Francesco Paolo and Wolford, Brooke N and Guan, Li and Varshney, Arushi and Albanus, Ricardo D'oliveira and Parker, Stephen C J and Narisu, Narisu and Chines, Peter S and Erdos, Michael R and Welch, Ryan P and Kinnunen, Leena and Saramies, Jouko and Sundvall, Jouko and Lakka, Timo A and Laakso, Markku and Tuomilehto, Jaakko and Koistinen, Heikki A and Stegle, Oliver and Boehnke, Michael and Birney, Ewan and Collins, Francis S},
  title = {Interactions between genetic variation and cellular environment in skeletal muscle gene expression},
  journal = {PLoS One},
  publisher = {Public Library of Science},
  year = {2018},
  doi = {10.1371/journal.pone.0195788}
}
</pre>
</div>
</li><li><div id="Knowles2018lacrosse"><em>Knowles DA, Bouchard G and Plevritis SK</em> (2019), <strong>"Sparse discriminative latent characteristics for predicting cancer drug sensitivity from genomic features"</strong>, PLoS computational biology. </div>
<div>[<a href="#bib_Knowles2018lacrosse" data-toggle="collapse">BibTeX</a>]</div>

<div id="bib_Knowles2018lacrosse" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2018lacrosse,
  author = {Knowles, David A and Bouchard, Gina and Plevritis, Sylvia K},
  title = {Sparse discriminative latent characteristics for predicting cancer drug sensitivity from genomic features},
  journal = {PLoS computational biology},
  year = {2019}
}
</pre>
</div>
</li><li><div id="Knowles2017gxe"><em>Knowles DA, Davis JR, Edgington H, Raj A, Fav&eacute; M-J, Zhu X, Potash JB, Weissman MM, Shi J, Levinson D, Awadalla P, Mostafavi S, Montgomery SB and Battle A</em> (2017), <strong>"Allele-specific expression reveals interactions between genetic variation and environment"</strong>, Nature Methods. </div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2017gxe">Abstract</a>][<a href="#bib_Knowles2017gxe" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/nmeth.4298" target="_blank">DOI</a>] [<a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4298.html" target="_blank">URL</a>] [<a href="http://biorxiv.org/content/early/2015/09/13/025874.abstract" target="_blank">Preprint</a>] [<a href="https://github.com/davidaknowles/eagle" target="_blank">Code</a>]</div>

<div id="abs_Knowles2017gxe" class="collapse">
	<b>Abstract</b>: Identifying interactions between genetics and the environment (GxE) remains challenging. We have developed EAGLE, a hierarchical Bayesian model for identifying GxE interactions based on association between environment and allele-specific expression (ASE). Combining RNA-sequencing of whole blood and extensive environmental annotations collected from 922 human individuals, we identified 35 GxE interactions, compared to only four using standard GxE testing. EAGLE provides new opportunities to identify GxE interactions using functional genomic data.
</div>
<div id="bib_Knowles2017gxe" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2017gxe,
  author = {Knowles, David A and Davis, Joe R and Edgington, Hilary and Raj, Anil and Fav&eacute;, Marie-Julie and Zhu, Xiaowei and Potash, James B and Weissman, Myrna M and Shi, Jianxin and Levinson, Doug and Awadalla, Philip and Mostafavi, Sara and Montgomery, Stephen B and Battle, Alexis},
  title = {Allele-specific expression reveals interactions between genetic variation and environment},
  journal = {Nature Methods},
  year = {2017},
  url = {http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4298.html},
  doi = {10.1038/nmeth.4298}
}
</pre>
</div>
</li><li><div id="Tung2017"><em>Tung P-Y, Blischak JD, Hsiao CJ, Knowles DA, Burnett JE, Pritchard JK and Gilad Y</em> (2017), <strong>"Batch effects and the effective design of single-cell gene expression studies"</strong>, Scientific Reports. </div>
<div>[<a data-toggle="collapse" href="#abs_Tung2017">Abstract</a>][<a href="#bib_Tung2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/srep39921" target="_blank">DOI</a>] [<a href="http://www.nature.com/articles/srep39921" target="_blank">URL</a>]</div>

<div id="abs_Tung2017" class="collapse">
	<b>Abstract</b>: Single cell RNA sequencing (scRNA-seq) can be used to characterize variation in gene expression levels at high resolution. However, the sources of experimental noise in scRNA-seq are not yet well understood. We investigated the technical variation associated with sample processing using the single cell Fluidigm C1 platform. To do so, we processed three C1 replicates from three human induced pluripotent stem cell (iPSC) lines. We added unique molecular identifiers (UMIs) to all samples, to account for amplification bias. We found that the major source of variation in the gene expression data was driven by genotype, but we also observed substantial variation between the technical replicates. We observed that the conversion of reads to molecules using the UMIs was impacted by both biological and technical variation, indicating that UMI counts are not an unbiased estimator of gene expression levels. Based on our results, we suggest a framework for effective scRNA-seq studies.
</div>
<div id="bib_Tung2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Tung2017,
  author = {Tung, Po-Yuan and Blischak, John D. and Hsiao, Chiaowen Joyce and Knowles, David A. and Burnett, Jonathan E. and Pritchard, Jonathan K. and Gilad, Yoav},
  title = {Batch effects and the effective design of single-cell gene expression studies},
  journal = {Scientific Reports},
  year = {2017},
  url = {http://www.nature.com/articles/srep39921},
  doi = {10.1038/srep39921}
}
</pre>
</div>
</li><li><div id="Calderon2017"><em>Calderon D, Bhaskar A, Knowles DA, Golan D, Raj T, Fu AQ and Pritchard JK</em> (2017), <strong>"Inferring Relevant Cell Types for Complex Traits by Using Single-Cell Gene Expression"</strong>, American Journal of Human Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Calderon2017">Abstract</a>][<a href="#bib_Calderon2017" data-toggle="collapse">BibTeX</a>] [<a href="http://www.cell.com/ajhg/abstract/S0002-9297(17)30378-6" target="_blank">URL</a>] [<a href="https://www.biorxiv.org/content/early/2017/05/10/136283" target="_blank">Preprint</a>] [<a href="https://cran.r-project.org/web/packages/rolypoly/index.html" target="_blank">Code</a>]</div>

<div id="abs_Calderon2017" class="collapse">
	<b>Abstract</b>: Previous studies have prioritized trait-relevant cell types by looking for an enrichment of GWAS signal within functional regions. However, these studies are limited in cell resolution by the lack of functional annotations from difficult-to-characterize or rare cell populations. Measurement of single-cell gene expression has become a popular method for characterizing novel cell types, and yet, hardly any work exists linking single-cell RNA-seq to phenotypes of interest. To address this deficiency, we present RolyPoly, a regression-based polygenic model that can prioritize trait-relevant cell types and genes from GWAS summary statistics and single-cell RNA-seq. We demonstrate RolyPoly's accuracy through simulation and validate previously known tissue-trait associations. We discover a significant association between microglia and late-onset Alzheimer's disease, and an association between oligodendrocytes and replicating fetal cortical cells with schizophrenia. Additionally, RolyPoly computes a trait-relevance score for each gene which reflects the importance of expression specific to a cell type. We found that differentially expressed genes in the prefrontal cortex of Alzheimer's patients were significantly enriched for highly ranked genes by RolyPoly gene scores. Overall, our method represents a powerful framework for understanding the effect of common variants on cell types contributing to complex traits.
</div>
<div id="bib_Calderon2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Calderon2017,
  author = {Calderon, Diego and Bhaskar, Anand and Knowles, David A and Golan, David and Raj, Towfique and Fu, Audrey Q and Pritchard, Jonathan K},
  title = {Inferring Relevant Cell Types for Complex Traits by Using Single-Cell Gene Expression},
  journal = {American Journal of Human Genetics},
  year = {2017},
  url = {http://www.cell.com/ajhg/abstract/S0002-9297(17)30378-6}
}
</pre>
</div>
</li><li><div id="Tsang2017"><em>Tsang EK, Abell NS, Li X, Anaya V, Karczewski KJ, Knowles DA, Sierra RG, Smith KS and Montgomery SB</em> (2017), <strong>"Small RNA sequencing in cells and exosomes identifies eQTLs and 14q32 as a region of active export"</strong>, G3 Genes|Genomes|Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Tsang2017">Abstract</a>][<a href="#bib_Tsang2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1534/g3.116.036137" target="_blank">DOI</a>] [<a href="http://g3journal.org/lookup/doi/10.1534/g3.116.036137" target="_blank">URL</a>]</div>

<div id="abs_Tsang2017" class="collapse">
	<b>Abstract</b>: Exosomes are small extracellular vesicles that carry heterogeneous cargo, including RNA, between cells. Increasing evidence suggests that exosomes are important mediators of intercellular communication and biomarkers of disease. Despite this, the variability of exosomal RNA between individuals has not been well quantified. To assess this variability, we sequenced the small RNA of cells and exosomes from a 17-member family. Across individuals, we show that selective export of miRNAs occurs not only at the level of specific transcripts, but that a cluster of 74 mature miRNAs on chromosome 14q32 is massively exported in exosomes while mostly absent from cells. We also observe more interindividual variability between exosomal samples than between cellular ones and identify four miRNA expression quantitative trait loci shared between cells and exosomes. Our findings indicate that genomically colocated miRNAs can be exported together and highlight the variability in exosomal miRNA levels between individuals as relevant for exosome use as diagnostics.
</div>
<div id="bib_Tsang2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Tsang2017,
  author = {Tsang, Emily K. and Abell, Nathan S. and Li, Xin and Anaya, Vanessa and Karczewski, Konrad J. and Knowles, David A. and Sierra, Raymond G. and Smith, Kevin S. and Montgomery, Stephen B.},
  title = {Small RNA sequencing in cells and exosomes identifies eQTLs and 14q32 as a region of active export},
  journal = {G3 Genes|Genomes|Genetics},
  year = {2017},
  url = {http://g3journal.org/lookup/doi/10.1534/g3.116.036137},
  doi = {10.1534/g3.116.036137}
}
</pre>
</div>
</li><li><div id="Becker2017"><em>Becker LA, Huang B, Bieri G, Ma R, Knowles DA, Jafar-Nejad P, Messing J, Kim HJ, Soriano A, Auburger G, Pulst SM, Taylor JP, Rigo F and Gitler AD</em> (2017), <strong>"Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice"</strong>, Nature. </div>
<div>[<a data-toggle="collapse" href="#abs_Becker2017">Abstract</a>][<a href="#bib_Becker2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/nature22038" target="_blank">DOI</a>] [<a href="http://www.nature.com/doifinder/10.1038/nature22038" target="_blank">URL</a>]</div>

<div id="abs_Becker2017" class="collapse">
	<b>Abstract</b>: Amyotrophic lateral sclerosis (ALS) is a rapidly progressing neurodegenerative disease that is characterized by motor neuron loss and that leads to paralysis and death 2--5 years after disease onset 1 . Nearly all patients with ALS have aggregates of the RNA-binding protein TDP-43 in their brains and spinal cords 2 , and rare mutations in the gene encoding TDP-43 can cause ALS 3 . There are no effective TDP-43-directed therapies for ALS or related TDP-43 proteinopathies, such as frontotemporal dementia. Antisense oligonucleotides (ASOs) and RNA-interference approaches are emerging as attractive therapeutic strategies in neurological diseases 4 . Indeed, treatment of a rat model of inherited ALS (caused by a mutation in Sod1) with ASOs against Sod1 has been shown to substantially slow disease progression 5 . However, as SOD1 mutations account for only around 2--5% of ALS cases, additional therapeutic strategies are needed. Silencing TDP-43 itself is probably not appropriate, given its critical cellular functions 1,6 . Here we present a promising alternative therapeutic strategy for ALS that involves targeting ataxin-2. A decrease in ataxin-2 suppresses TDP-43 toxicity in yeast and flies 7 , and intermediate-length polyglutamine expansions in the ataxin-2 gene increase risk of ALS 7,8 . We used two independent approaches to test whether decreasing ataxin-2 levels could mitigate disease in a mouse model of TDP-43 proteinopathy 9 . First, we crossed ataxin-2 knockout mice with TDP-43 (also known as TARDBP) transgenic mice. The decrease in ataxin-2 reduced aggregation of TDP-43, markedly increased survival and improved motor function. Second, in a more therapeutically applicable approach, we administered ASOs targeting ataxin-2 to the central nervous system of TDP-43 transgenic mice. This single treatment markedly extended survival. Because TDP-43 aggregation is a component of nearly all cases of ALS 6 , targeting ataxin-2 could represent a broadly effective therapeutic strategy. To test the hypothesis that a decrease in ataxin-2 levels can res-cue neurodegenerative phenotypes caused by TDP-43 accumula-tion, we first used a genetic approach. There are several transgenic mouse lines that express wild-type or mutant TDP-43, using various strategies 10
</div>
<div id="bib_Becker2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Becker2017,
  author = {Becker, Lindsay A. and Huang, Brenda and Bieri, Gregor and Ma, Rosanna and Knowles, David A. and Jafar-Nejad, Paymaan and Messing, James and Kim, Hong Joo and Soriano, Armand and Auburger, Georg and Pulst, Stefan M. and Taylor, J. Paul and Rigo, Frank and Gitler, Aaron D.},
  title = {Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice},
  journal = {Nature},
  year = {2017},
  url = {http://www.nature.com/doifinder/10.1038/nature22038},
  doi = {10.1038/nature22038}
}
</pre>
</div>
</li><li><div id="Davis2016eigenmt"><em>Davis JR, Fresard L, Knowles DA, Pala M, Bustamante CD, Battle A and Montgomery SB</em> (2016), <strong>"An Efficient Multiple-Testing Adjustment for eQTL Studies that Accounts for Linkage Disequilibrium between Variants"</strong>, The American Journal of Human Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Davis2016eigenmt">Abstract</a>][<a href="#bib_Davis2016eigenmt" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.ajhg.2015.11.021" target="_blank">DOI</a>] [<a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00492-9" target="_blank">URL</a>] [<a href="http://montgomerylab.stanford.edu/resources/eigenMT/eigenMT.html" target="_blank">Code</a>]</div>

<div id="abs_Davis2016eigenmt" class="collapse">
	<b>Abstract</b>: Methods for multiple-testing correction in local expression quantitative trait locus (cis-eQTL) studies are a trade-off between statistical power and computational efficiency. Bonferroni correction, though computationally trivial, is overly conservative and fails to account for linkage disequilibrium between variants. Permutation-based methods are more powerful, though computationally far more intensive. We present an alternative correction method called eigenMT, which runs over 500 times faster than permutations and has adjusted p values that closely approximate empirical ones. To achieve this speed while also maintaining the accuracy of permutation-based methods, we estimate the effective number of independent variants tested for association with a particular gene, termed Meff, by using the eigenvalue decomposition of the genotype correlation matrix. We employ a regularized estimator of the correlation matrix to ensure Meff is robust and yields adjusted p values that closely approximate p values from permutations. Finally, using a common genotype matrix, we show that eigenMT can be applied with even greater efficiency to studies across tissues or conditions. Our method provides a simpler, more efficient approach to multiple-testing correction than existing methods and fits within existing pipelines for eQTL discovery.
</div>
<div id="bib_Davis2016eigenmt" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Davis2016eigenmt,
  author = {Davis, Joe R. and Fresard, Laure and Knowles, David A. and Pala, Mauro and Bustamante, Carlos D. and Battle, Alexis and Montgomery, Stephen B.},
  title = {An Efficient Multiple-Testing Adjustment for eQTL Studies that Accounts for Linkage Disequilibrium between Variants},
  journal = {The American Journal of Human Genetics},
  year = {2016},
  url = {http://www.cell.com/ajhg/abstract/S0002-9297(15)00492-9},
  doi = {10.1016/j.ajhg.2015.11.021}
}
</pre>
</div>
</li><li><div id="Kukurba2015"><em>Kukurba KR, Parsana P, Balliu B, Smith KS, Zappala Z, Knowles DA, Fav&eacute; M-J, Davis JR, Li X, Zhu X, Potash JB, Weissman MM, Shi J, Kundaje A, Levinson DF, Awadalla P, Mostafavi S, Battle A and Montgomery SB</em> (2016), <strong>"Impact of the X chromosome and sex on regulatory variation"</strong>, Genome Research. </div>
<div>[<a data-toggle="collapse" href="#abs_Kukurba2015">Abstract</a>][<a href="#bib_Kukurba2015" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/gr.197897.115" target="_blank">DOI</a>] [<a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.197897.115" target="_blank">URL</a>]</div>

<div id="abs_Kukurba2015" class="collapse">
	<b>Abstract</b>: The X chromosome, with its unique mode of inheritance, contributes to differences between the sexes at a molecular level, including sex-specific gene expression and sex-specific impact of genetic variation. We have conducted an analysis of the impact of both sex and the X chromosome on patterns of gene expression identified through transcriptome sequencing of whole blood from 922 individuals. We identified that genes on the X chromosome are more likely to have sex-specific expression compared to the autosomal genes. Furthermore, we identified a depletion of regulatory variants on the X chromosome, especially among genes under high selective constraint. In contrast, we discovered an enrichment of sex-specific regulatory variants on the X chromosome. To resolve the molecular mechanisms underlying such effects, we generated and connected sex-specific chromatin accessibility to sex-specific expression and regulatory variation. As sex-specific regulatory variants can inform sex differences in genetic disease prevalence, we have integrated our data with genome-wide association study data for multiple immune traits and to identify traits with significant sex biases. Together, our study provides genome-wide insight into how the X chromosome and sex shape human gene regulation and disease.
</div>
<div id="bib_Kukurba2015" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Kukurba2015,
  author = {Kukurba, Kimberly R. and Parsana, Princy and Balliu, Brunilda and Smith, Kevin S. and Zappala, Zachary and Knowles, David A. and Fav&eacute;, Marie-Julie and Davis, Joe R. and Li, Xin and Zhu, Xiaowei and Potash, James B. and Weissman, Myrna M. and Shi, Jianxin and Kundaje, Anshul and Levinson, Douglas F. and Awadalla, Philip and Mostafavi, Sara and Battle, Alexis and Montgomery, Stephen B.},
  title = {Impact of the X chromosome and sex on regulatory variation},
  journal = {Genome Research},
  publisher = {Cold Spring Harbor Labs Journals},
  year = {2016},
  url = {http://genome.cshlp.org/lookup/doi/10.1101/gr.197897.115},
  doi = {10.1101/gr.197897.115}
}
</pre>
</div>
</li><li><div id="LeafCutter"><em>Li* YI, Knowles* DA, Humphrey J, Barbeira AN, Dickinson SP, Im HK and Pritchard JK</em> (2017), <strong>"Annotation-free quantification of RNA splicing using LeafCutter"</strong>, Nature Genetics.  *These authors contributed equally to this work.</div>
<div>[<a data-toggle="collapse" href="#abs_LeafCutter">Abstract</a>][<a href="#bib_LeafCutter" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/s41588-017-0004-9" target="_blank">DOI</a>] [<a href="https://www.nature.com/articles/s41588-017-0004-9" target="_blank">URL</a>] [<a href="https://github.com/davidaknowles/leafcutter" target="_blank">Code</a>]</div>

<div id="abs_LeafCutter" class="collapse">
	<b>Abstract</b>: The excision of introns from pre-mRNA is an essential step in mRNA processing. We developed LeafCutter to study sample and population variation in intron splicing. LeafCutter identifies variable intron splicing events from short-read RNA-seq data and finds alternative splicing events of high complexity. Our approach obviates the need for transcript annotations and overcomes the challenges in estimating relative isoform or exon usage in complex splicing events. LeafCutter can be used both for detecting differential splicing between sample groups, and for mapping splicing quantitative trait loci (sQTLs). Compared to contemporary methods, we find over three times more sQTLs, many of which help us ascribe molecular effects to disease-associated variants. LeafCutter is fast, easy to use, and available at https://github.com/davidaknowles/
</div>
<div id="bib_LeafCutter" class="collapse">
<b>BibTeX</b>:
<pre>
@article{LeafCutter,
  author = {Li*, Yang I. and Knowles*, David A. and Humphrey, Jack and Barbeira, Alvaro N. and Dickinson, Scott P. and Im, Hae Kyung and Pritchard, Jonathan K.},
  title = {Annotation-free quantification of RNA splicing using LeafCutter},
  journal = {Nature Genetics},
  year = {2017},
  url = {https://www.nature.com/articles/s41588-017-0004-9},
  doi = {10.1038/s41588-017-0004-9}
}
</pre>
</div>
</li><li><div id="Li2016splicing"><em>Li YI, van de Geijn B, Raj A, Knowles DA, Petti AA, Golan D, Gilad Y and Pritchard JK</em> (2016), <strong>"RNA splicing is a primary link between genetic variation and disease."</strong>, Science. </div>
<div>[<a data-toggle="collapse" href="#abs_Li2016splicing">Abstract</a>][<a href="#bib_Li2016splicing" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1126/science.aad9417" target="_blank">DOI</a>] [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27126046" target="_blank">URL</a>]</div>

<div id="abs_Li2016splicing" class="collapse">
	<b>Abstract</b>: Noncoding variants play a central role in the genetics of complex traits, but we still lack a full understanding of the molecular pathways through which they act. We quantified the contribution of cis-acting genetic effects at all major stages of gene regulation from chromatin to proteins, in Yoruba lymphoblastoid cell lines (LCLs). About 65% of expression quantitative trait loci (eQTLs) have primary effects on chromatin, whereas the remaining eQTLs are enriched in transcribed regions. Using a novel method, we also detected 2893 splicing QTLs, most of which have little or no effect on gene-level expression. These splicing QTLs are major contributors to complex traits, roughly on a par with variants that affect gene expression levels. Our study provides a comprehensive view of the mechanisms linking genetic variation to variation in human gene regulation.
</div>
<div id="bib_Li2016splicing" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Li2016splicing,
  author = {Li, Yang I and van de Geijn, Bryce and Raj, Anil and Knowles, David A and Petti, Allegra A and Golan, David and Gilad, Yoav and Pritchard, Jonathan K},
  title = {RNA splicing is a primary link between genetic variation and disease.},
  journal = {Science},
  publisher = {American Association for the Advancement of Science},
  year = {2016},
  url = {http://www.ncbi.nlm.nih.gov/pubmed/27126046},
  doi = {10.1126/science.aad9417}
}
</pre>
</div>
</li><li><div id="Kukurba2014deleterious"><em>Kukurba KR, Zhang R, Li X, Smith KS, Knowles DA, How Tan M, Piskol R, Lek M, Snyder M, MacArthur DG, Li JB and Montgomery SB</em> (2014), <strong>"Allelic Expression of Deleterious Protein-Coding Variants across Human Tissues"</strong>, PLoS Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Kukurba2014deleterious">Abstract</a>][<a href="#bib_Kukurba2014deleterious" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pgen.1004304" target="_blank">DOI</a>] [<a href="http://dx.plos.org/10.1371/journal.pgen.1004304" target="_blank">URL</a>]</div>

<div id="abs_Kukurba2014deleterious" class="collapse">
	<b>Abstract</b>: Personal exome and genome sequencing provides access to loss-of-function and rare deleterious alleles whose interpretation is expected to provide insight into individual disease burden. However, for each allele, accurate interpretation of its effect will depend on both its penetrance and the trait's expressivity. In this regard, an important factor that can modify the effect of a pathogenic coding allele is its level of expression; a factor which itself characteristically changes across tissues. To better inform the degree to which pathogenic alleles can be modified by expression level across multiple tissues, we have conducted exome, RNA and deep, targeted allele-specific expression (ASE) sequencing in ten tissues obtained from a single individual. By combining such data, we report the impact of rare and common loss-of-function variants on allelic expression exposing stronger allelic bias for rare stop-gain variants and informing the extent to which rare deleterious coding alleles are consistently expressed across tissues. This study demonstrates the potential importance of transcriptome data to the interpretation of pathogenic protein-coding variants.
</div>
<div id="bib_Kukurba2014deleterious" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Kukurba2014deleterious,
  author = {Kukurba, Kimberly R. and Zhang, Rui and Li, Xin and Smith, Kevin S. and Knowles, David A. and How Tan, Meng and Piskol, Robert and Lek, Monkol and Snyder, Michael and MacArthur, Daniel G. and Li, Jin Billy and Montgomery, Stephen B.},
  title = {Allelic Expression of Deleterious Protein-Coding Variants across Human Tissues},
  journal = {PLoS Genetics},
  publisher = {Public Library of Science},
  year = {2014},
  url = {http://dx.plos.org/10.1371/journal.pgen.1004304},
  doi = {10.1371/journal.pgen.1004304}
}
</pre>
</div>
</li><li><div id="Li2014rare"><em>Li X, Battle A, Karczewski KJ, Zappala Z, Knowles DA, Smith KS, Kukurba KR, Wu E, Simon N and Montgomery SB</em> (2014), <strong>"Transcriptome sequencing of a large human family identifies the impact of rare noncoding variants."</strong>, American Journal of Human Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Li2014rare">Abstract</a>][<a href="#bib_Li2014rare" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.ajhg.2014.08.004" target="_blank">DOI</a>] [<a href="http://www.cell.com/article/S0002929714003486/fulltext" target="_blank">URL</a>]</div>

<div id="abs_Li2014rare" class="collapse">
	<b>Abstract</b>: Recent and rapid human population growth has led to an excess of rare genetic variants that are expected to contribute to an individual's genetic burden of disease risk. To date, much of the focus has been on rare protein-coding variants, for which potential impact can be estimated from the genetic code, but determining the impact of rare noncoding variants has been more challenging. To improve our understanding of such variants, we combined high-quality genome sequencing and RNA sequencing data from a 17-individual, three-generation family to contrast expression quantitative trait loci (eQTLs) and splicing quantitative trait loci (sQTLs) within this family to eQTLs and sQTLs within a population sample. Using this design, we found that eQTLs and sQTLs with large effects in the family were enriched with rare regulatory and splicing variants (minor allele frequency 0.01). They were also more likely to influence essential genes and genes involved in complex disease. In addition, we tested the capacity of diverse noncoding annotation to predict the impact of rare noncoding variants. We found that distance to the transcription start site, evolutionary constraint, and epigenetic annotation were considerably more informative for predicting the impact of rare variants than for predicting the impact of common variants. These results highlight that rare noncoding variants are important contributors to individual gene-expression profiles and further demonstrate a significant capability for genomic annotation to predict the impact of rare noncoding variants..
</div>
<div id="bib_Li2014rare" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Li2014rare,
  author = {Li, Xin and Battle, Alexis and Karczewski, Konrad J. and Zappala, Zach and Knowles, David A. and Smith, Kevin S. and Kukurba, Kim R. and Wu, Eric and Simon, Noah and Montgomery, Stephen B.},
  title = {Transcriptome sequencing of a large human family identifies the impact of rare noncoding variants.},
  journal = {American Journal of Human Genetics},
  publisher = {Elsevier},
  year = {2014},
  url = {http://www.cell.com/article/S0002929714003486/fulltext},
  doi = {10.1016/j.ajhg.2014.08.004}
}
</pre>
</div>
</li><li><div id="Glass2013muther"><em>Glass D, Vi&ntilde;uela A, Davies MN, Ramasamy A, Parts L, Knowles DA, Brown AA, Hedman AK, Small KS, Buil A, Grundberg E, Nica AC, Meglio P, Nestle FO, Ryten M, Durbin R, McCarthy MI, Deloukas P, Dermitzakis ET, Weale ME, Bataille V and Spector TD</em> (2013), <strong>"Gene expression changes with age in skin, adipose tissue, blood and brain."</strong>, Genome biology. </div>
<div>[<a data-toggle="collapse" href="#abs_Glass2013muther">Abstract</a>][<a href="#bib_Glass2013muther" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1186/gb-2013-14-7-r75" target="_blank">DOI</a>] [<a href="http://genomebiology.com/2013/14/7/R75" target="_blank">URL</a>]</div>

<div id="abs_Glass2013muther" class="collapse">
	<b>Abstract</b>: BACKGROUND: Previous studies have demonstrated that gene expression levels change with age. These changes are hypothesized to influence the aging rate of an individual. We analyzed gene expression changes with age in abdominal skin, subcutaneous adipose tissue and lymphoblastoid cell lines in 856 female twins in the age range of 39-85 years. Additionally, we investigated genotypic variants involved in genotype-by-age interactions to understand how the genomic regulation of gene expression alters with age. RESULTS: Using a linear mixed model, differential expression with age was identified in 1,672 genes in skin and 188 genes in adipose tissue. Only two genes expressed in lymphoblastoid cell lines showed significant changes with age. Genes significantly regulated by age were compared with expression profiles in 10 brain regions from 100 postmortem brains aged 16 to 83 years. We identified only one age-related gene common to the three tissues. There were 12 genes that showed differential expression with age in both skin and brain tissue and three common to adipose and brain tissues. CONCLUSIONS: Skin showed the most age-related gene expression changes of all the tissues investigated, with many of the genes being previously implicated in fatty acid metabolism, mitochondrial activity, cancer and splicing. A significant proportion of age-related changes in gene expression appear to be tissue-specific with only a few genes sharing an age effect in expression across tissues. More research is needed to improve our understanding of the genetic influences on aging and the relationship with age-related diseases.
</div>
<div id="bib_Glass2013muther" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Glass2013muther,
  author = {Glass, Daniel and Vi&ntilde;uela, Ana and Davies, Matthew N and Ramasamy, Adaikalavan and Parts, Leopold and Knowles, David A. and Brown, Andrew A and Hedman, Asa K and Small, Kerrin S and Buil, Alfonso and Grundberg, Elin and Nica, Alexandra C and Meglio, Paoladi and Nestle, Frank O and Ryten, Mina and Durbin, Richard and McCarthy, Mark I and Deloukas, Panagiotis and Dermitzakis, Emmanouil T and Weale, Michael E and Bataille, Veronique and Spector, Tim D},
  title = {Gene expression changes with age in skin, adipose tissue, blood and brain.},
  journal = {Genome biology},
  year = {2013},
  url = {http://genomebiology.com/2013/14/7/R75},
  doi = {10.1186/gb-2013-14-7-r75}
}
</pre>
</div>
</li><li><div id="Grundberg2012"><em>Grundberg E, Small KS, Hedman AK, Nica AC, Buil A, Keildson S, Bell JT, Yang T-P, Meduri E, Barrett A, Nisbett J, Sekowska M, Wilk A, Shin S-Y, Glass D, Travers M, Min JL, Knowles DA, Ring S, Ho K, Thorleifsson G, Kong A, Thorsteindottir U, Ainali C, Dimas AS, Hassanali N, Ingle C, Krestyaninova M, Lowe CE, Di Meglio P, Montgomery SB, Parts L, Potter S, Surdulescu G, Tsaprouni L, Tsoka S, Bataille V, Durbin R, Nestle FO, O'Rahilly S, Soranzo N, Lindgren CM, Zondervan KT, Ahmadi KR, Schadt EE, Stefansson K, Smith GD, McCarthy MI, Deloukas P, Dermitzakis ET and Spector TD</em> (2012), <strong>"Mapping cis- and trans-regulatory effects across multiple tissues in twins."</strong>, Nature Genetics. </div>
<div>[<a data-toggle="collapse" href="#abs_Grundberg2012">Abstract</a>][<a href="#bib_Grundberg2012" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/ng.2394" target="_blank">DOI</a>] [<a href="http://dx.doi.org/10.1038/ng.2394" target="_blank">URL</a>]</div>

<div id="abs_Grundberg2012" class="collapse">
	<b>Abstract</b>: Sequence-based variation in gene expression is a key driver of disease risk. Common variants regulating expression in cis have been mapped in many expression quantitative trait locus (eQTL) studies, typically in single tissues from unrelated individuals. Here, we present a comprehensive analysis of gene expression across multiple tissues conducted in a large set of mono- and dizygotic twins that allows systematic dissection of genetic (cis and trans) and non-genetic effects on gene expression. Using identity-by-descent estimates, we show that at least 40% of the total heritable cis effect on expression cannot be accounted for by common cis variants, a finding that reveals the contribution of low-frequency and rare regulatory variants with respect to both transcriptional regulation and complex trait susceptibility. We show that a substantial proportion of gene expression heritability is trans to the structural gene, and we identify several replicating trans variants that act predominantly in a tissue-restricted manner and may regulate the transcription of many genes.
</div>
<div id="bib_Grundberg2012" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Grundberg2012,
  author = {Grundberg, Elin and Small, Kerrin S and Hedman, Asa K and Nica, Alexandra C and Buil, Alfonso and Keildson, Sarah and Bell, Jordana T and Yang, Tsun-Po and Meduri, Eshwar and Barrett, Amy and Nisbett, James and Sekowska, Magdalena and Wilk, Alicja and Shin, So-Youn and Glass, Daniel and Travers, Mary and Min, Josine L and Knowles, David A. and Ring, Sue and Ho, Karen and Thorleifsson, Gudmar and Kong, Augustine and Thorsteindottir, Unnur and Ainali, Chrysanthi and Dimas, Antigone S and Hassanali, Neelam and Ingle, Catherine and Krestyaninova, Maria and Lowe, Christopher E and Di Meglio, Paola and Montgomery, Stephen B and Parts, Leopold and Potter, Simon and Surdulescu, Gabriela and Tsaprouni, Loukia and Tsoka, Sophia and Bataille, Veronique and Durbin, Richard and Nestle, Frank O and O'Rahilly, Stephen and Soranzo, Nicole and Lindgren, Cecilia M and Zondervan, Krina T and Ahmadi, Kourosh R and Schadt, Eric E and Stefansson, Kari and Smith, George Davey and McCarthy, Mark I and Deloukas, Panos and Dermitzakis, Emmanouil T and Spector, Tim D.},
  title = {Mapping cis- and trans-regulatory effects across multiple tissues in twins.},
  journal = {Nature Genetics},
  year = {2012},
  url = {http://dx.doi.org/10.1038/ng.2394},
  doi = {10.1038/ng.2394}
}
</pre>
</div>
</li><li><div id="Schone2011"><em>Sch&ouml;ne C, Venner A, Knowles DA, Karnani MM and Burdakov D</em> (2011), <strong>"Dichotomous cellular properties of mouse orexin/hypocretin neurons."</strong>, The Journal of Physiology. </div>
<div>[<a data-toggle="collapse" href="#abs_Schone2011">Abstract</a>][<a href="#bib_Schone2011" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1113/jphysiol.2011.208637" target="_blank">DOI</a>] [<a href="http://jp.physoc.org/content/early/2011/04/11/jphysiol.2011.208637.abstract" target="_blank">URL</a>]</div>

<div id="abs_Schone2011" class="collapse">
	<b>Abstract</b>: Hypothalamic hypocretin/orexin (Hcrt/Orx) neurons recently emerged as critical regulators of sleep--wake cycles, reward seeking and body energy balance. However, at the level of cellular and network properties, it remains unclear whether Hcrt/Orx neurons are one homogeneous population, or whether there are several distinct types of Hcrt/Orx cells. Here, we collated diverse structural and functional information about individual Hcrt/Orx neurons in mouse brain slices, by combining patch-clamp analysis of spike firing, membrane currents and synaptic inputs with confocal imaging of cell shape and subsequent 3-dimensional Sholl analysis of dendritic architecture. Statistical cluster analysis of intrinsic firing properties revealed that Hcrt/Orx neurons fall into two distinct types. These two cell types also differ in the complexity of their dendritic arbour, the strength of AMPA and GABAA receptor-mediated synaptic drive that they receive, and the density of low-threshold, 4-aminopyridine-sensitive, transient K+ current. Our results provide quantitative evidence that, at the cellular level, the mouse Hcrt/Orx system is composed of two classes of neurons with different firing properties, morphologies and synaptic input organization.
</div>
<div id="bib_Schone2011" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Schone2011,
  author = {Sch&ouml;ne, Cornelia and Venner, Anne and Knowles, David A. and Karnani, Mahesh M and Burdakov, Denis},
  title = {Dichotomous cellular properties of mouse orexin/hypocretin neurons.},
  journal = {The Journal of Physiology},
  year = {2011},
  url = {http://jp.physoc.org/content/early/2011/04/11/jphysiol.2011.208637.abstract},
  doi = {10.1113/jphysiol.2011.208637}
}
</pre>
</div>
</li><li><div id="Movassagh2011a"><em>Movassagh M, Choy M-K, Knowles DA, Cordeddu L, Haider S, Down T, Siggens L, Vujic A, Simeoni I, Penkett C, Goddard M, Lio P, Bennett M and Foo R</em> (2011), <strong>"Distinct Epigenomic Features in End-Stage Failing Human Hearts"</strong>, Circulation, American Heart Association. </div>
<div>[<a data-toggle="collapse" href="#abs_Movassagh2011a">Abstract</a>][<a href="#bib_Movassagh2011a" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1161/CIRCULATIONAHA.111.040071" target="_blank">DOI</a>] [<a href="http://circ.ahajournals.org/content/early/2011/10/24/CIRCULATIONAHA.111.040071.abstract" target="_blank">URL</a>]</div>

<div id="abs_Movassagh2011a" class="collapse">
	<b>Abstract</b>: BACKGROUND: The epigenome refers to marks on the genome, including DNA methylation and histone modifications, that regulate the expression of underlying genes. A consistent profile of gene expression changes in end-stage cardiomyopathy led us to hypothesize that distinct global patterns of the epigenome may also exist.  METHODS AND RESULTS: We constructed genome-wide maps of DNA methylation and histone-3 lysine-36 trimethylation (H3K36me3) enrichment for cardiomyopathic and normal human hearts. More than 506 Mb sequences per library were generated by high-throughput sequencing, allowing us to assign methylation scores to ≈28 million CG dinucleotides in the human genome. DNA methylation was significantly different in promoter CpG islands, intragenic CpG islands, gene bodies, and H3K36me3-enriched regions of the genome. DNA methylation differences were present in promoters of upregulated genes but not downregulated genes. H3K36me3 enrichment itself was also significantly different in coding regions of the genome. Specifically, abundance of RNA transcripts encoded by the DUX4 locus correlated to differential DNA methylation and H3K36me3 enrichment. In vitro, Dux gene expression was responsive to a specific inhibitor of DNA methyltransferase, and Dux siRNA knockdown led to reduced cell viability.  CONCLUSIONS: Distinct epigenomic patterns exist in important DNA elements of the cardiac genome in human end-stage cardiomyopathy. The epigenome may control the expression of local or distal genes with critical functions in myocardial stress response. If epigenomic patterns track with disease progression, assays for the epigenome may be useful for assessing prognosis in heart failure. Further studies are needed to determine whether and how the epigenome contributes to the development of cardiomyopathy.
</div>
<div id="bib_Movassagh2011a" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Movassagh2011a,
  author = {Movassagh, Mehregan and Choy, Mun-Kit and Knowles, David A and Cordeddu, Lina and Haider, Syed and Down, Thomas and Siggens, Lee and Vujic, Ana and Simeoni, Ilenia and Penkett, Chris and Goddard, Martin and Lio, Pietro and Bennett, Martin and Foo, Roger},
  title = {Distinct Epigenomic Features in End-Stage Failing Human Hearts},
  journal = {Circulation, American Heart Association},
  year = {2011},
  url = {http://circ.ahajournals.org/content/early/2011/10/24/CIRCULATIONAHA.111.040071.abstract},
  doi = {10.1161/CIRCULATIONAHA.111.040071}
}
</pre>
</div>
</li><li><div id="Glass2010"><em>Glass D, Parts L, Knowles D, Aviv A and Spector TD</em> (2010), <strong>"No correlation between childhood maltreatment and telomere length."</strong>, Biological psychiatry. </div>
<div>[<a href="#bib_Glass2010" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.biopsych.2010.02.026" target="_blank">DOI</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2930212/" target="_blank">URL</a>]</div>

<div id="bib_Glass2010" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Glass2010,
  author = {Glass, Daniel and Parts, Leopold and Knowles, David and Aviv, Abraham and Spector, Tim D},
  title = {No correlation between childhood maltreatment and telomere length.},
  journal = {Biological psychiatry},
  year = {2010},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2930212/},
  doi = {10.1016/j.biopsych.2010.02.026}
}
</pre>
</div>
</li></ol>