<ol>
<li><div id="Banerjee2025-nb"><em>Banerjee S, O'Connell S, Colbert SMC, Mullins N and Knowles DA</em> (2025), <strong>"Convex approaches to isolate the shared and distinct genetic structures of subphenotypes in heterogeneous complex traits"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Banerjee2025-nb">Abstract</a>][<a href="#bib_Banerjee2025-nb" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2025.04.15.25325870v1.abstract" target="_blank">URL</a>]</div>

<div id="abs_Banerjee2025-nb" class="collapse">
	<b>Abstract</b>: Groups of complex diseases, such as coronary heart diseases, neuropsychiatric disorders, and cancers, often display overlapping clinical symptoms and pharmacological treatments. The shared associations of genetic variants across diseases has the potential to explain their underlying biological processes, but this remains poorly understood. To address this, we model the matrix of summary statistics of trait-associated genetic variants as the sum of a low-rank component - representing shared biological processes - and a sparse component, representing unique processes and arbitrarily corrupted or contaminated components. We introduce Clorinn, an open-source Python software that uses convex optimization algorithms to recover these components by minimizing a weighted combination of the nuclear norm and of the L1 norm. Among others, Clorinn provides two significant benefits: (a) Convex optimization guarantees reproducibility of the components, and (b) The low-rank ``uncorrupted'' matrix allows robust singular value decomposition (SVD) and principal component analysis (PCA), which are otherwise highly sensitive to outliers and noise in the input matrix. In extensive simulations, we observe that Clorinn outperforms state-of-the-art approaches in capturing the shared latent factors across phenotypes. We apply Clorinn to estimate 200 latent factors from GWAS summary data of 2,110 phenotypes measured in European-ancestry Pan-UK BioBank individuals N = 420,531 and 14 psychiatric disorders.
</div>
<div id="bib_Banerjee2025-nb" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Banerjee2025-nb,
  author = {Banerjee, Saikat and O'Connell, Shane and Colbert, Sarah M C and Mullins, Niamh and Knowles, David A},
  title = {Convex approaches to isolate the shared and distinct genetic structures of subphenotypes in heterogeneous complex traits},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2025.04.15.25325870v1.abstract}
}
</pre>
</div>
</li><li><div id="Naito2025-xn"><em>Naito T, Hirata K, Jang B, Lakhani CM, Buonfiglioli A, Lee W-P, Valladares O, Wang L-S, Okada Y, Won H-H, Papapetrou EP, Marro SG, Knowles DA and Raj T</em> (2025), <strong>"Mosaic chromosomal alterations in blood are associated with an increased risk of Alzheimer's disease"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Naito2025-xn">Abstract</a>][<a href="#bib_Naito2025-xn" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2025.05.29.25328544.abstract" target="_blank">URL</a>]</div>

<div id="abs_Naito2025-xn" class="collapse">
	<b>Abstract</b>: Mosaic chromosomal alterations (mCAs) in blood, a form of clonal hematopoiesis, have been linked to various diseases, but their role in Alzheimer's disease (AD) remains unclear. We analyzed blood whole-genome sequencing (WGS) data from 24,049 individuals in the Alzheimer's Disease Sequencing Project and found that autosomal mCAs were significantly associated with increased AD risk (odds ratio = 1.27; P = 1.3e-5). This association varied by ancestry, mCA subtype, APOE ε4 allele status, and chromosomal location. Using matched blood WGS and brain single-nucleus RNA-seq data, we identified microglia-annotated cells in the brain carrying the same mCAs found in blood. These findings suggest that blood mCAs may contribute to AD pathogenesis, potentially through infiltration into the brain and influencing local immune response.
</div>
<div id="bib_Naito2025-xn" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Naito2025-xn,
  author = {Naito, Tatsuhiko and Hirata, Kosei and Jang, Beomjin and Lakhani, Chirag M and Buonfiglioli, Alice and Lee, Wan-Ping and Valladares, Otto and Wang, Li-San and Okada, Yukinori and Won, Hong-Hee and Papapetrou, Eirini P and Marro, Samuele G and Knowles, David A and Raj, Towfique},
  title = {Mosaic chromosomal alterations in blood are associated with an increased risk of Alzheimer's disease},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2025.05.29.25328544.abstract}
}
</pre>
</div>
</li><li><div id="Becker2025-cd"><em>Becker LA, Quinodoz SA, Comi TJ, Kimchi O, Knowles DA and Brangwynne CP</em> (2025), <strong>"Genome-wide mapping of mesoscale neuronal RNA organization and condensation"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Becker2025-cd">Abstract</a>][<a href="#bib_Becker2025-cd" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2025.04.19.649570v1.abstract" target="_blank">URL</a>]</div>

<div id="abs_Becker2025-cd" class="collapse">
	<b>Abstract</b>: Subcellular RNA organization can affect critical cellular functions. However, our understanding of RNA microenvironments, particularly biomolecular condensates, remains limited, largely due to a lack of technologies to comprehensively interrogate mesoscale RNA organization. Here, we adapt Split-Pool Recognition of Interactions by Tag Extension to map micron-scale RNA-RNA spatial proximity genome-wide across cell regions (RNA-SPRITE). Deploying RNA-SPRITE, we find extensive, conserved organization of mature mRNAs, with increased colocalization between mRNAs that share RNA-binding protein (RBP) motifs or encode functionally related proteins. Both effects are especially strong in dendrites and axons, suggesting prevalent mRNA co-regulation. Moreover, mRNAs with less compact folding, lower translation efficiency, and specific RBP motifs are more likely to be in RNA-rich condensates. However, perturbations that broadly dissolve or enhance condensation reveal that RBP motif and encoded protein-mediated colocalizations largely remain intact, independent of condensation. These results demonstrate the power of RNA-SPRITE in revealing critical aspects of RNA’s functional organization. In Brief: Unbiased, genome-wide maps of RNA-RNA mesoscale spatial proximity uncover extensive subcellular organization and its governing principles. Highlights: RNA-SPRITE reveals micron-scale RNA colocalization genome-wide across cell regions; mRNA colocalization specificity is driven by shared motifs and encoded protein function; mRNAs with less compact folding, lower translation efficiency, and distinct protein-binding motifs are more likely to be in condensates; Neurites have a particularly high degree of sequence and function-dependent mRNA organization.
</div>
<div id="bib_Becker2025-cd" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Becker2025-cd,
  author = {Becker, Lindsay A and Quinodoz, Sofia A and Comi, Troy J and Kimchi, Ofer and Knowles, David A and Brangwynne, Clifford P},
  title = {Genome-wide mapping of mesoscale neuronal RNA organization and condensation},
  journal = {bioRxiv},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2025.04.19.649570v1.abstract}
}
</pre>
</div>
</li><li><div id="Meng2025-pq"><em>Meng L, Yu A, Huang H, Meng J, Yoo Y, Schaukowitch K, Janas JA and Knowles DA</em> (2025), <strong>"Oncogenic mutations disrupt lineage differentiation via epigenetic perturbations"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Meng2025-pq">Abstract</a>][<a href="#bib_Meng2025-pq" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649974.abstract" target="_blank">URL</a>]</div>

<div id="abs_Meng2025-pq" class="collapse">
	<b>Abstract</b>: Understanding lineage differentiation is an ongoing challenge in the field of stem cell research. We acquired eight gain-of-function (GOF) oncogenic mutations associated with DNA methylation alteration from the TCGA datasets. After introducing each oncogene into wild-type stem cells separately, we found that the mutant stem cells are difficult to reprogram to iPSCs and terminally differentiate to induced neuronal cells, and retain proliferation capacity in the process of differentiation. This findings suggests that oncogenic mutations impair the lineage differentiation. After analyzing the resulting DNA methylation profiles, we identified many novel lineage differentiation genes in mutant cells showing DNA methylation alteration compared with wild-type cells, which is consistent with their methylation states in low-grade gliomas (LGGs) compared with control brain. Collectively, our results indicate that oncogenic mutations in stem cells are associated with DNA methylation changes for genes related to multipotency maintenance, differentiation resistance, and tumorigenesis, therefore, displaying impairment in lineage commitment.
</div>
<div id="bib_Meng2025-pq" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Meng2025-pq,
  author = {Meng, Lingjun and Yu, Alice and Huang, Huanwei and Meng, Jodie and Yoo, Yongjin and Schaukowitch, Katie and Janas, Justyna A and Knowles, David A},
  title = {Oncogenic mutations disrupt lineage differentiation via epigenetic perturbations},
  journal = {bioRxiv},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2025.04.22.649974.abstract}
}
</pre>
</div>
</li><li><div id="Su2025-by"><em>Su J, Qu Y, Schertzer M, Yang H, Jiang J, Lhakhang T, Nelson TM, Park S, Lai Q, Fu X, Choi S-W, Knowles DA and Rabadan R</em> (2025), <strong>"A computational framework for mapping isoform landscape and regulatory mechanisms from spatial transcriptomics data"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Su2025-by">Abstract</a>][<a href="#bib_Su2025-by" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651907.abstract" target="_blank">URL</a>]</div>

<div id="abs_Su2025-by" class="collapse">
	<b>Abstract</b>: Transcript diversity including splicing and alternative 3’end usage is crucial for cellular identity and adaptation, yet its spatial coordination remains poorly understood. Here, we present SPLISOSM (SpatiaL ISOform Statistical Modeling), a computational framework for detecting isoform-resolution patterns from spatial transcriptomics data. SPLISOSM leverages multivariate testing to account for spot- and isoform-level dependencies, demonstrating robust and theoretically grounded performance on sparse data. In the mouse brain, we identify over 1,000 spatially variable transcript diversity events, primarily in synaptic signaling pathways linked to neuropsychiatric disorders, and uncover both known and novel regulatory relationships with region-specific RNA binding proteins. We further show that these patterns are evolutionarily conserved between mouse and human prefrontal cortex. Analysis of human glioblastoma highlights pervasive transcript diversity in antigen presentation and adhesion genes associated with specific microenvironmental conditions. Together, we present a comprehensive spatial splicing analysis in the brain under normal and neoplastic conditions.HighlightsMultivariate tests of spatial variability and association for transcript usageConserved isoform variability in synaptic pathways across mouse and human brainsCoordinated RNA binding protein expression drives region-specific regulationTumor microenvironment shapes spatial transcript landscapes in human glioma
</div>
<div id="bib_Su2025-by" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Su2025-by,
  author = {Su, Jiayu and Qu, Yiming and Schertzer, Megan and Yang, Haochen and Jiang, Jiahao and Lhakhang, Tenzin and Nelson, Theodore M and Park, Stella and Lai, Qiliang and Fu, Xi and Choi, Seung-Won and Knowles, David A and Rabadan, Raul},
  title = {A computational framework for mapping isoform landscape and regulatory mechanisms from spatial transcriptomics data},
  journal = {bioRxiv},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2025.05.02.651907.abstract}
}
</pre>
</div>
</li><li><div id="Zweig2025-cd"><em>Zweig A, Lin Z, Azizi E and Knowles DA</em> (2025), <strong>"Towards identifiability of interventional stochastic differential equations"</strong>, arXiv [cs.LG]. </div>
<div>[<a data-toggle="collapse" href="#abs_Zweig2025-cd">Abstract</a>][<a href="#bib_Zweig2025-cd" data-toggle="collapse">BibTeX</a>] [<a href="http://arxiv.org/abs/2505.15987" target="_blank">URL</a>]</div>

<div id="abs_Zweig2025-cd" class="collapse">
	<b>Abstract</b>: We study identifiability of stochastic differential equation (SDE) models under multiple interventions. Our results give the first provable bounds for unique recovery of SDE parameters given samples from their stationary distributions. We give tight bounds on the number of necessary interventions for linear SDEs, and upper bounds for nonlinear SDEs in the small noise regime. We experimentally validate the recovery of true parameters in synthetic data, and motivated by our theoretical results, demonstrate the advantage of parameterizations with learnable activation functions.
</div>
<div id="bib_Zweig2025-cd" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Zweig2025-cd,
  author = {Zweig, Aaron and Lin, Zaikang and Azizi, Elham and Knowles, David A},
  title = {Towards identifiability of interventional stochastic differential equations},
  journal = {arXiv [cs.LG]},
  year = {2025},
  url = {http://arxiv.org/abs/2505.15987}
}
</pre>
</div>
</li><li><div id="Schertzer2025-pv"><em>Schertzer MD, Park SH, Su J, Sheynkman GM and Knowles DA</em> (2025), <strong>"Perplexity as a metric for isoform diversity in the human transcriptome"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Schertzer2025-pv">Abstract</a>][<a href="#bib_Schertzer2025-pv" data-toggle="collapse">BibTeX</a>] [<a href="http://dx.doi.org/10.1101/2025.07.02.662769" target="_blank">URL</a>]</div>

<div id="abs_Schertzer2025-pv" class="collapse">
	<b>Abstract</b>: Long-read sequencing (LRS) has revealed a far greater diversity of RNA isoforms than earlier technologies, increasing the critical need to determine which, and how many, isoforms per gene are biologically meaningful. To define the space of relevant isoforms from LRS, many existing analysis pipelines rely on arbitrary expression cutoffs, but a single threshold cannot accommodate the broad variability in isoform complexity across genes, cell-types, and disease states captured by LRS. To address this, we propose using perplexity, an interpretable measure derived from entropy, that quantifies the effective number of isoforms per gene based on the full, unfiltered isoform ratio distribution. Calculating perplexity for 124 ENCODE4 PacBio LRS datasets spanning 55 human cell types, we show that it provides intuitive assessments of isoform diversity and captures uncertainty across genes with varying complexity. Perplexity can be calculated at multiple gene regulatory levels, from transcript to protein, to compare how isoform diversity is reduced across stages of gene expression. On average, genes have an ORF-level perplexity of 2.1, indicating production of two distinct protein isoforms. We extended this analysis to evaluate expression variation across tissues and identified 4,593 ORFs across 3,102 genes with moderate to extreme tissue-specificity. We propose perplexity as a consistent, quantitative metric for interpreting isoform diversity across genes, cell types, and disease states. All results are compiled into a community resource to enable cross-study comparisons of novel isoforms.
</div>
<div id="bib_Schertzer2025-pv" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Schertzer2025-pv,
  author = {Schertzer, Megan D and Park, Stella H and Su, Jiayu and Sheynkman, Gloria M and Knowles, David A},
  title = {Perplexity as a metric for isoform diversity in the human transcriptome},
  journal = {bioRxiv},
  year = {2025},
  url = {http://dx.doi.org/10.1101/2025.07.02.662769}
}
</pre>
</div>
</li><li><div id="Moses2025-kh"><em>Moses AM, Stajich JE, Gasch AP and Knowles DA</em> (2025), <strong>"Inferring fungal cis-regulatory networks from genome sequences"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Moses2025-kh">Abstract</a>][<a href="#bib_Moses2025-kh" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2025.02.27.640643.abstract" target="_blank">URL</a>]</div>

<div id="abs_Moses2025-kh" class="collapse">
	<b>Abstract</b>: Gene expression patterns are determined to a large extent by transcription factor binding to non-coding regulatory regions in the genome. However, gene expression cannot yet be systematically predicted from genome sequences, in part because non-functional matches to the sequence patterns (motifs) recognized by transcription factors (TFs) occur frequently throughout the genome. Large-scale functional genomics data for many TFs has enabled characterization of regulatory networks in experimentally accessible cells such as budding yeast. Beyond yeast, fungi are important industrial organisms and pathogens, but large-scale functional data is only sporadically available. Uncharacterized regulatory networks control key pathways and gene expression programs associated with fungal phenotypes. Here we explore a sequence-only approach to inferring regulatory networks by leveraging the 100s of genomes now available for many clades of fungi. We use gene orthology as the learning signal to infer interpretable, TF motif-based representations of non-coding regulatory regions, thus scaling comparative genomics beyond evolutionary comparisons where these regions can be aligned. We show that similarity of promoters in our motif-based representation predicts gene co-expression, and that we can infer known and novel regulatory connections in diverse fungi. Our new predictions include a pathway for recombination inC. albicans and pathways for mating and an RNAi immune response inNeurospora. Taken together, our results indicate that specific hypotheses about transcriptional regulation in fungi can be obtained for many genes from genome sequence analysis alone.
</div>
<div id="bib_Moses2025-kh" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Moses2025-kh,
  author = {Moses, Alan M and Stajich, Jason E and Gasch, Audrey P and Knowles, David A},
  title = {Inferring fungal cis-regulatory networks from genome sequences},
  journal = {bioRxiv},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2025.02.27.640643.abstract}
}
</pre>
</div>
</li><li><div id="Jang2025-vj"><em>Jang B, Bp K, Tokolyi A, Cuddleston WH, Ravi A, Jung S-H, Naito T, Kim B, Seo Kim M, Cho M, Park M-S, Rosen M, Blanchard J, Humphrey J, Knowles DA, Won H-H and Raj T</em> (2025), <strong>"SingleBrain: A meta-analysis of single-nucleus eQTLs linking genetic risk to brain disorders"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Jang2025-vj">Abstract</a>][<a href="#bib_Jang2025-vj" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2025.03.06.25323424.abstract" target="_blank">URL</a>]</div>

<div id="abs_Jang2025-vj" class="collapse">
	<b>Abstract</b>: Most genetic risk variants for neurological diseases are located in non-coding regulatory regions, where they may often act as expression quantitative trait loci (eQTLs), modulating gene expression and influencing disease susceptibility. However, eQTL studies in bulk brain tissue or specific cell types lack the resolution to capture the brain's cellular diversity. Single-nucleus RNA sequencing (snRNA-seq) offers high-resolution mapping of eQTLs across diverse brain cell types. Here, we performed a meta-analysis, SingleBrain, integrating publicly available snRNA-seq and genotype data from four cohorts, totaling 5.8 million nuclei from 983 individuals. We mapped cis-eQTLs across major brain cell types and subtypes and employed statistical colocalization and Mendelian randomization to identify genes mediating neurological disease risk. We observed up to a 10-fold increase in cis-eQTLs compared to previous studies and uncovered novel cell type-specific genes linked to Alzheimer's disease, Parkinson's disease, and schizophrenia that were previously undetectable in bulk tissue analyses. Additionally, we prioritized putative causal variants for each locus through fine-mapping and integration with cell type-specific enhancer and promoter regulatory elements. SingleBrain represents a comprehensive single-cell eQTL resource, advancing insights into the genetic regulation of brain disorders.
</div>
<div id="bib_Jang2025-vj" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Jang2025-vj,
  author = {Jang, Beomjin and Bp, Kailash and Tokolyi, Alex and Cuddleston, Winston H and Ravi, Ashvin and Jung, Sang-Hyuk and Naito, Tatsuhiko and Kim, Beomsu and Seo Kim, Min and Cho, Minyoung and Park, Mi-So and Rosen, Mikaela and Blanchard, Joel and Humphrey, Jack and Knowles, David A and Won, Hong-Hee and Raj, Towfique},
  title = {SingleBrain: A meta-analysis of single-nucleus eQTLs linking genetic risk to brain disorders},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2025.03.06.25323424.abstract}
}
</pre>
</div>
</li><li><div id="Lin2025-gg"><em>Lin Z, Chang S, Zweig A, Kang M, Azizi E and Knowles DA</em> (2025), <strong>"Interpretable neural ODEs for gene regulatory network discovery under perturbations"</strong>, arXiv [cs.LG]. </div>
<div>[<a data-toggle="collapse" href="#abs_Lin2025-gg">Abstract</a>][<a href="#bib_Lin2025-gg" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/2501.02409" target="_blank">URL</a>]</div>

<div id="abs_Lin2025-gg" class="collapse">
	<b>Abstract</b>: Modern high-throughput biological datasets with thousands of perturbations provide the opportunity for large-scale discovery of causal graphs that represent the regulatory interactions between genes. Differentiable causal graphical models have been proposed to infer a gene regulatory network (GRN) from large scale interventional datasets, capturing the causal gene regulatory relationships from genetic perturbations. However, existing models are limited in their expressivity and scalability while failing to address the dynamic nature of biological processes such as cellular differentiation. We propose PerturbODE, a novel framework that incorporates biologically informative neural ordinary differential equations (neural ODEs) to model cell state trajectories under perturbations and derive the causal GRN from the neural ODE's parameters. We demonstrate PerturbODE's efficacy in trajectory prediction and GRN inference across simulated and real over-expression datasets.
</div>
<div id="bib_Lin2025-gg" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Lin2025-gg,
  author = {Lin, Zaikang and Chang, Sei and Zweig, Aaron and Kang, Minseo and Azizi, Elham and Knowles, David A},
  title = {Interpretable neural ODEs for gene regulatory network discovery under perturbations},
  journal = {arXiv [cs.LG]},
  year = {2025},
  url = {https://arxiv.org/abs/2501.02409}
}
</pre>
</div>
</li><li><div id="Lakhani2025-fs"><em>Lakhani CM, Lin J-ST, Das A, Naito T, Raj T and Knowles DA</em> (2025), <strong>"Integration of deep learning annotations with functional genomics improves identification of causal Alzheimer's disease variants"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Lakhani2025-fs">Abstract</a>][<a href="#bib_Lakhani2025-fs" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2025.03.07.25323578v1" target="_blank">URL</a>]</div>

<div id="abs_Lakhani2025-fs" class="collapse">
	<b>Abstract</b>: Genetic variants associated with Alzheimer's disease (AD) through genome-wide association studies (GWAS) are challenging to interpret because most lie in non-coding regions of the genome. Here, a method was developed that integrates deep learning variant effect prediction (DL-VEP) scores from Enformer, DeepSea, and ChromBPNet models with cell-type specific regulatory annotations to improve fine-mapping of causal AD variants. Using stratified linkage disequilibrium score regression, the largest proportion of heritability among brain cell types was found in microglia and the integration of Enformer microglia-based transcription factor DL-VEP scores and microglia ChromBPNet models further localizes the AD polygenic signal allowing for improved fine-mapping of causal AD variants. Functionally-informed fine-mapping using these annotations discovers 101 AD risk variants compared to 87 identified by statistical fine-mapping alone. Importantly, polygenic risk scores derived from these fine-mapped variants show improved cross-ancestry performance in both African American (AUC=0.532 vs. 0.521 for SUSIE) and Hispanic (AUC=0.552 vs. 0.535 for SUSIE) populations, while maintaining strong performance in European populations (AUC=0.620 vs. 0.615 for SUSIE). Through detailed analysis of the PICALM/EED locus, we demonstrate how our approach disentangles causal variants in regions of high linkage disequilibrium and predicts specific molecular mechanisms, such as disruption of PU.1 transcription factor binding, with the posterior inclusion probability of this variant increasing from 0.55 using SUSIE to 0.90 with our method. Our results provide a framework for leveraging deep learning annotations to improve identification of causal disease variants and enhance polygenic risk prediction across diverse populations.
</div>
<div id="bib_Lakhani2025-fs" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Lakhani2025-fs,
  author = {Lakhani, Chirag M and Lin, Jui-Shan T and Das, Anjali and Naito, Tatsuhiko and Raj, Towfique and Knowles, David A},
  title = {Integration of deep learning annotations with functional genomics improves identification of causal Alzheimer's disease variants},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2025.03.07.25323578v1}
}
</pre>
</div>
</li></ol>