<ol>
<li><div id="Fernando2024-pw"><em>Fernando MB, Fan Y, Zhang Y, Tokolyi A, Murphy AN, Kammourh S, Michael Deans PJ, Ghorbani S, Onatzevitch R, Pero A, Padilla C, Williams S, Flaherty EK, Prytkova IA, Cao L, Knowles DA, Fang G, Slesinger PA and Brennand KJ</em> (2025), <strong>"Phenotypic complexities of rare heterozygous neurexin-1 deletions"</strong>, bioRxiv (to appear in Nature 2025). </div>
<div>[<a data-toggle="collapse" href="#abs_Fernando2024-pw">Abstract</a>][<a href="#bib_Fernando2024-pw" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2023.10.28.564543v3" target="_blank">URL</a>]</div>

<div id="abs_Fernando2024-pw" class="collapse">
	<b>Abstract</b>: Given the large number of genes significantly associated with risk for neuropsychiatric disorders, a critical unanswered question is the extent to which diverse mutations --sometimes impacting the same gene-- will require tailored therapeutic strategies. Here we consider this in the context of rare neuropsychiatric disorder-associated copy number variants (2p16.3) resulting in heterozygous deletions in NRXN1, a pre-synaptic cell adhesion protein that serves as a critical synaptic organizer in the brain. Complex patterns of NRXN1 alternative splicing are fundamental to establishing diverse neurocircuitry, vary between the cell types of the brain, and are differentially impacted by unique (non-recurrent) deletions. We contrast the cell-type-specific impact of patient-specific mutations in NRXN1 using human induced pluripotent stem cells, finding that perturbations in NRXN1 splicing result in divergent cell-type-specific synaptic outcomes. Via distinct loss-of-function (LOF) and gain-of-function (GOF) mechanisms, NRXN1 +/- deletions cause decreased synaptic activity in glutamatergic neurons, yet increased synaptic activity in GABAergic neurons. Reciprocal isogenic manipulations causally demonstrate that aberrant splicing drives these changes in synaptic activity. For NRXN1 deletions, and perhaps more broadly, precision medicine will require stratifying patients based on whether their gene mutations act through LOF or GOF mechanisms, in order to achieve individualized restoration of NRXN1 isoform repertoires by increasing wildtype, or ablating mutant isoforms. Given the increasing number of mutations predicted to engender both LOF and GOF mechanisms in brain disorders, our findings add nuance to future considerations of precision medicine.
</div>
<div id="bib_Fernando2024-pw" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Fernando2024-pw,
  author = {Fernando, Michael B and Fan, Yu and Zhang, Yanchun and Tokolyi, Alex and Murphy, Aleta N and Kammourh, Sarah and Michael Deans, P J and Ghorbani, Sadaf and Onatzevitch, Ryan and Pero, Adriana and Padilla, Christopher and Williams, Sarah and Flaherty, Erin K and Prytkova, Iya A and Cao, Lei and Knowles, David A and Fang, Gang and Slesinger, Paul A and Brennand, Kristen J},
  title = {Phenotypic complexities of rare heterozygous neurexin-1 deletions},
  journal = {bioRxiv (to appear in Nature 2025)},
  year = {2025},
  url = {https://www.biorxiv.org/content/10.1101/2023.10.28.564543v3}
}
</pre>
</div>
</li><li><div id="Terrey2024-xi"><em>Terrey M, Krivoshein G, Adamson SI, Arystarkhova E, Anderson L, Szwec J, McKee S, Jones H, Perkins S, Selvam V, Piec P-A, Chhaya D, Dehn A, Zuberi A, Murray SA, Morsci NS, Sweadner KJ, Knowles DA, Tolner EA, van den Maagdenberg AMJM and Lutz CM</em> (2024), <strong>"Alternating hemiplegia of childhood associated mutations in Atp1a3 reveal diverse neurological alterations in mice"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Terrey2024-xi">Abstract</a>][<a href="#bib_Terrey2024-xi" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/2024.12.24.630241" target="_blank">URL</a>]</div>

<div id="abs_Terrey2024-xi" class="collapse">
	<b>Abstract</b>: Pathogenic variants in the neuronal Na+/K+ ATPase transmembrane ion transporter (ATP1A3) cause a spectrum of neurological disorders including alternating hemiplegia of childhood (AHC). The most common de novo pathogenic variants in AHC are p.D801N (&nbsp;40% of patients) and p.E815K (&nbsp;25% of patients), which lead to early mortality by spontaneous death in mice. Nevertheless, knowledge of the development of clinically relevant neurological phenotypes without the obstacle of premature death, is critical for the identification of pathophysiological mechanisms and ultimately, for the testing of therapeutic strategies in disease models. Here, we used hybrid vigor attempting to mitigate the fragility of AHC mice and then, performed behavioral, electrophysiological, biochemical, and molecular testing to comparatively analyze mice that carry either of two most common AHC patient observed variants in the Atp1a3 gene. Collectively, our data reveal the presence but also the differential impact of the p.D801N and p.E815 variants on disease relevant alterations such as spontaneous and stress-induced paroxysmal episodes, motor function, behavioral and neurophysiological activity, and neuroinflammation. Our alternate AHC mouse models with their phenotypic deficits open novel avenues for the investigation of disease biology and therapeutic testing for ATP1A3 research.
</div>
<div id="bib_Terrey2024-xi" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Terrey2024-xi,
  author = {Terrey, Markus and Krivoshein, Georgii and Adamson, Scott I and Arystarkhova, Elena and Anderson, Laura and Szwec, John and McKee, Shelby and Jones, Holly and Perkins, Sara and Selvam, Vijay and Piec, Pierre-Alexandre and Chhaya, Dweet and Dehn, Ari and Zuberi, Aamir and Murray, Stephen A and Morsci, Natalia S and Sweadner, Kathleen J and Knowles, David A and Tolner, Else A and van den Maagdenberg, Arn M J M and Lutz, Cathleen M},
  title = {Alternating hemiplegia of childhood associated mutations in Atp1a3 reveal diverse neurological alterations in mice},
  journal = {bioRxiv},
  year = {2024},
  url = {https://doi.org/10.1101/2024.12.24.630241}
}
</pre>
</div>
</li><li><div id="O-Connell2024-lh"><em>O'Connell S, Brown BC, Cannon DM, Broin PÓ, Knowles DA, Parker N, Alnæs D, Westlye LT, Banerjee S, Nabulsi L, Corley E, Andreassen OA and Mullins N</em> (2024), <strong>"Deriving Mendelian Randomization-based causal networks of brain imaging phenotypes and bipolar disorder"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_O-Connell2024-lh">Abstract</a>][<a href="#bib_O-Connell2024-lh" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2024.12.12.24318953v1" target="_blank">URL</a>]</div>

<div id="abs_O-Connell2024-lh" class="collapse">
	<b>Abstract</b>: Neuroanatomical variation in individuals with bipolar disorder (BD) has been previously described in observational studies. However, the causal dynamics of these relationships remain unexplored. We performed Mendelian Randomization of 297 structural and functional neuroimaging phenotypes from the UK BioBank and BD using genome-wide association study summary statistics. We found 28 significant causal relationship pairs after multiple testing corrections containing BD as a term, 27 of which described neuroimaging phenotype effects on BD. We applied an inverse sparse regression algorithm to estimate the direct effect of phenotypes conditional on all other causal effects, finding that white matter tract phenotypes have larger absolute effects on BD than vice versa. We found that white matter phenotypes have significantly larger out-degrees than non-white matter tract phenotypes, and that the effect of neuroimaging variation on BD is larger than vice versa. Our results provide support for the hypothesis that neuroanatomical variation, specifically in white matter tracts such as the superior and inferior longitudinal fasciculi, is a cause rather than a consequence of BD.
</div>
<div id="bib_O-Connell2024-lh" class="collapse">
<b>BibTeX</b>:
<pre>
@article{O-Connell2024-lh,
  author = {O'Connell, Shane and Brown, Brielin C and Cannon, Dara M and Broin, Pilib Ó and Knowles, David A and Parker, Nadine and Alnæs, Dag and Westlye, Lars T and Banerjee, Saikat and Nabulsi, Leila and Corley, Emma and Andreassen, Ole A and Mullins, Niamh},
  title = {Deriving Mendelian Randomization-based causal networks of brain imaging phenotypes and bipolar disorder},
  journal = {medRxiv},
  year = {2024},
  url = {https://www.medrxiv.org/content/10.1101/2024.12.12.24318953v1}
}
</pre>
</div>
</li><li><div id="Jang2025-vj"><em>Jang B, Bp K, Tokolyi A, Cuddleston WH, Ravi A, Jung S-H, Naito T, Kim B, Seo Kim M, Cho M, Park M-S, Rosen M, Blanchard J, Humphrey J, Knowles DA, Won H-H and Raj T</em> (2025), <strong>"SingleBrain: A meta-analysis of single-nucleus eQTLs linking genetic risk to brain disorders"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Jang2025-vj">Abstract</a>][<a href="#bib_Jang2025-vj" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2025.03.06.25323424.abstract" target="_blank">URL</a>]</div>

<div id="abs_Jang2025-vj" class="collapse">
	<b>Abstract</b>: Most genetic risk variants for neurological diseases are located in non-coding regulatory regions, where they may often act as expression quantitative trait loci (eQTLs), modulating gene expression and influencing disease susceptibility. However, eQTL studies in bulk brain tissue or specific cell types lack the resolution to capture the brain's cellular diversity. Single-nucleus RNA sequencing (snRNA-seq) offers high-resolution mapping of eQTLs across diverse brain cell types. Here, we performed a meta-analysis, SingleBrain, integrating publicly available snRNA-seq and genotype data from four cohorts, totaling 5.8 million nuclei from 983 individuals. We mapped cis-eQTLs across major brain cell types and subtypes and employed statistical colocalization and Mendelian randomization to identify genes mediating neurological disease risk. We observed up to a 10-fold increase in cis-eQTLs compared to previous studies and uncovered novel cell type-specific genes linked to Alzheimer's disease, Parkinson's disease, and schizophrenia that were previously undetectable in bulk tissue analyses. Additionally, we prioritized putative causal variants for each locus through fine-mapping and integration with cell type-specific enhancer and promoter regulatory elements. SingleBrain represents a comprehensive single-cell eQTL resource, advancing insights into the genetic regulation of brain disorders.
</div>
<div id="bib_Jang2025-vj" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Jang2025-vj,
  author = {Jang, Beomjin and Bp, Kailash and Tokolyi, Alex and Cuddleston, Winston H and Ravi, Ashvin and Jung, Sang-Hyuk and Naito, Tatsuhiko and Kim, Beomsu and Seo Kim, Min and Cho, Minyoung and Park, Mi-So and Rosen, Mikaela and Blanchard, Joel and Humphrey, Jack and Knowles, David A and Won, Hong-Hee and Raj, Towfique},
  title = {SingleBrain: A meta-analysis of single-nucleus eQTLs linking genetic risk to brain disorders},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2025.03.06.25323424.abstract}
}
</pre>
</div>
</li><li><div id="Lin2025-gg"><em>Lin Z, Chang S, Zweig A, Kang M, Azizi E and Knowles DA</em> (2025), <strong>"Interpretable neural ODEs for gene regulatory network discovery under perturbations"</strong>, arXiv [cs.LG]. </div>
<div>[<a data-toggle="collapse" href="#abs_Lin2025-gg">Abstract</a>][<a href="#bib_Lin2025-gg" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/2501.02409" target="_blank">URL</a>]</div>

<div id="abs_Lin2025-gg" class="collapse">
	<b>Abstract</b>: Modern high-throughput biological datasets with thousands of perturbations provide the opportunity for large-scale discovery of causal graphs that represent the regulatory interactions between genes. Differentiable causal graphical models have been proposed to infer a gene regulatory network (GRN) from large scale interventional datasets, capturing the causal gene regulatory relationships from genetic perturbations. However, existing models are limited in their expressivity and scalability while failing to address the dynamic nature of biological processes such as cellular differentiation. We propose PerturbODE, a novel framework that incorporates biologically informative neural ordinary differential equations (neural ODEs) to model cell state trajectories under perturbations and derive the causal GRN from the neural ODE's parameters. We demonstrate PerturbODE's efficacy in trajectory prediction and GRN inference across simulated and real over-expression datasets.
</div>
<div id="bib_Lin2025-gg" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Lin2025-gg,
  author = {Lin, Zaikang and Chang, Sei and Zweig, Aaron and Kang, Minseo and Azizi, Elham and Knowles, David A},
  title = {Interpretable neural ODEs for gene regulatory network discovery under perturbations},
  journal = {arXiv [cs.LG]},
  year = {2025},
  url = {https://arxiv.org/abs/2501.02409}
}
</pre>
</div>
</li><li><div id="Das2025-uy"><em>Das A, Lakhani C, Terwagne C, Lin J-ST, Naito T, Raj T and Knowles DA</em> (2025), <strong>"Leveraging functional annotations to map rare variants associated with Alzheimer's disease with gruyere"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Das2025-uy">Abstract</a>][<a href="#bib_Das2025-uy" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2024.12.06.24318577v2" target="_blank">URL</a>]</div>

<div id="abs_Das2025-uy" class="collapse">
	<b>Abstract</b>: The increasing availability of whole-genome sequencing (WGS) has begun to elucidate the contribution of rare variants (RVs), both coding and non-coding, to complex disease. Multiple RV association tests are available to study the relationship between genotype and phenotype, but most are restricted to per-gene models and do not fully leverage the availability of variant-level functional annotations. We propose Genome-wide Rare Variant EnRichment Evaluation (gruyere), a Bayesian probabilistic model that complements existing methods by learning global, trait-specific weights for functional annotations to improve variant prioritization. We apply gruyere to WGS data from the Alzheimer's Disease (AD) Sequencing Project, consisting of 7,966 cases and 13,412 controls, to identify AD-associated genes and annotations. Growing evidence suggests that disruption of microglial regulation is a key contributor to AD risk, yet existing methods have not had sufficient power to examine rare non-coding effects that incorporate such cell-type specific information. To address this gap, we 1) use predicted enhancer and promoter regions in microglia and other potentially relevant cell types (oligodendrocytes, astrocytes, and neurons) to define per-gene non-coding RV test sets and 2) include cell-type specific variant effect predictions (VEPs) as functional annotations. gruyere identifies 15 significant genetic associations not detected by other RV methods and finds deep learning-based VEPs for splicing, transcription factor binding, and chromatin state are highly predictive of functional non-coding RVs. Our study establishes a novel and robust framework incorporating functional annotations, coding RVs, and cell-type associated non-coding RVs, to perform genome-wide association tests, uncovering AD-relevant genes and annotations.
</div>
<div id="bib_Das2025-uy" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Das2025-uy,
  author = {Das, Anjali and Lakhani, Chirag and Terwagne, Chloé and Lin, Jui-Shan T and Naito, Tatsuhiko and Raj, Towfique and Knowles, David A},
  title = {Leveraging functional annotations to map rare variants associated with Alzheimer's disease with gruyere},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2024.12.06.24318577v2}
}
</pre>
</div>
</li><li><div id="Lakhani2025-fs"><em>Lakhani CM, Lin J-ST, Das A, Naito T, Raj T and Knowles DA</em> (2025), <strong>"Integration of deep learning annotations with functional genomics improves identification of causal Alzheimer's disease variants"</strong>, medRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Lakhani2025-fs">Abstract</a>][<a href="#bib_Lakhani2025-fs" data-toggle="collapse">BibTeX</a>] [<a href="https://www.medrxiv.org/content/10.1101/2025.03.07.25323578v1" target="_blank">URL</a>]</div>

<div id="abs_Lakhani2025-fs" class="collapse">
	<b>Abstract</b>: Genetic variants associated with Alzheimer's disease (AD) through genome-wide association studies (GWAS) are challenging to interpret because most lie in non-coding regions of the genome. Here, a method was developed that integrates deep learning variant effect prediction (DL-VEP) scores from Enformer, DeepSea, and ChromBPNet models with cell-type specific regulatory annotations to improve fine-mapping of causal AD variants. Using stratified linkage disequilibrium score regression, the largest proportion of heritability among brain cell types was found in microglia and the integration of Enformer microglia-based transcription factor DL-VEP scores and microglia ChromBPNet models further localizes the AD polygenic signal allowing for improved fine-mapping of causal AD variants. Functionally-informed fine-mapping using these annotations discovers 101 AD risk variants compared to 87 identified by statistical fine-mapping alone. Importantly, polygenic risk scores derived from these fine-mapped variants show improved cross-ancestry performance in both African American (AUC=0.532 vs. 0.521 for SUSIE) and Hispanic (AUC=0.552 vs. 0.535 for SUSIE) populations, while maintaining strong performance in European populations (AUC=0.620 vs. 0.615 for SUSIE). Through detailed analysis of the PICALM/EED locus, we demonstrate how our approach disentangles causal variants in regions of high linkage disequilibrium and predicts specific molecular mechanisms, such as disruption of PU.1 transcription factor binding, with the posterior inclusion probability of this variant increasing from 0.55 using SUSIE to 0.90 with our method. Our results provide a framework for leveraging deep learning annotations to improve identification of causal disease variants and enhance polygenic risk prediction across diverse populations.
</div>
<div id="bib_Lakhani2025-fs" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Lakhani2025-fs,
  author = {Lakhani, Chirag M and Lin, Jui-Shan T and Das, Anjali and Naito, Tatsuhiko and Raj, Towfique and Knowles, David A},
  title = {Integration of deep learning annotations with functional genomics improves identification of causal Alzheimer's disease variants},
  journal = {medRxiv},
  year = {2025},
  url = {https://www.medrxiv.org/content/10.1101/2025.03.07.25323578v1}
}
</pre>
</div>
</li><li><div id="Brown2023-uc"><em>Brown BC, Morris JA, Lappalainen T and Knowles DA</em> (2023), <strong>"Large-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits"</strong>, bioRxiv. , pp. 2023.10.13.562293.</div>
<div>[<a data-toggle="collapse" href="#abs_Brown2023-uc">Abstract</a>][<a href="#bib_Brown2023-uc" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2023.10.13.562293v1" target="_blank">URL</a>]</div>

<div id="abs_Brown2023-uc" class="collapse">
	<b>Abstract</b>: Inference of directed biological networks is an important but notoriously challenging problem. We introduce inverse sparse regression (inspre) , an approach to learning causal networks that leverages large-scale intervention-response data. Applied to 788 genes from the genome-wide perturb-seq dataset, inspre helps elucidate the network architecture of blood traits.
</div>
<div id="bib_Brown2023-uc" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Brown2023-uc,
  author = {Brown, Brielin C and Morris, John A and Lappalainen, Tuuli and Knowles, David A},
  title = {Large-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits},
  journal = {bioRxiv},
  year = {2023},
  pages = {2023.10.13.562293},
  url = {https://www.biorxiv.org/content/10.1101/2023.10.13.562293v1}
}
</pre>
</div>
</li><li><div id="Schertzer2023-lb"><em>Schertzer MD, Stirn A, Isaev K, Pereira L, Das A, Harbison C, Park SH, Wessels H-H, Sanjana NE and Knowles DA</em> (2023), <strong>"Cas13d-mediated isoform-specific RNA knockdown with a unified computational and experimental toolbox"</strong>, bioRxiv. , pp. 2023.09.12.557474.</div>
<div>[<a data-toggle="collapse" href="#abs_Schertzer2023-lb">Abstract</a>][<a href="#bib_Schertzer2023-lb" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2023.09.12.557474v1" target="_blank">URL</a>]</div>

<div id="abs_Schertzer2023-lb" class="collapse">
	<b>Abstract</b>: Alternative splicing is an essential mechanism for diversifying proteins, in which mature RNA isoforms produce proteins with potentially distinct functions. Two major challenges in characterizing the cellular function of isoforms are the lack of experimental methods to specifically and efficiently modulate isoform expression and computational tools for complex experimental design. To address these gaps, we developed and methodically tested a strategy which pairs the RNA-targeting CRISPR/Cas13d system with guide RNAs that span exon-exon junctions in the mature RNA. We performed a high-throughput essentiality screen, quantitative RT-PCR assays, and PacBio long read sequencing to affirm our ability to specifically target and robustly knockdown individual RNA isoforms. In parallel, we provide computational tools for experimental design and screen analysis. Considering all possible splice junctions annotated in GENCODE for multi-isoform genes and our gRNA efficacy predictions, we estimate that our junction-centric strategy can uniquely target up to 89% of human RNA isoforms, including 50,066 protein-coding and 11,415 lncRNA isoforms. Importantly, this specificity spans all splicing and transcriptional events, including exon skipping and inclusion, alternative 5' and 3' splice sites, and alternative starts and ends.
</div>
<div id="bib_Schertzer2023-lb" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Schertzer2023-lb,
  author = {Schertzer, Megan D and Stirn, Andrew and Isaev, Keren and Pereira, Laura and Das, Anjali and Harbison, Claire and Park, Stella H and Wessels, Hans-Hermann and Sanjana, Neville E and Knowles, David A},
  title = {Cas13d-mediated isoform-specific RNA knockdown with a unified computational and experimental toolbox},
  journal = {bioRxiv},
  year = {2023},
  pages = {2023.09.12.557474},
  url = {https://www.biorxiv.org/content/10.1101/2023.09.12.557474v1}
}
</pre>
</div>
</li><li><div id="Su2022"><em>Su J, Reynier J-B, Fu X, Zhong G, Jiang J, Escalante RS, Wang Y, Izar B, Knowles DA and Rabadan R</em> (2022), <strong>"A Unified Modular Framework to Incorporate Structural Dependency in Spatial Omics Data"</strong>, bioRxiv. , pp. 2022.10.25.513785.</div>
<div>[<a data-toggle="collapse" href="#abs_Su2022">Abstract</a>][<a href="#bib_Su2022" data-toggle="collapse">BibTeX</a>] [<a href="https://www.biorxiv.org/content/10.1101/2022.10.25.513785v1.abstract" target="_blank">URL</a>]</div>

<div id="abs_Su2022" class="collapse">
	<b>Abstract</b>: Spatial omics technologies, such as spatial transcriptomics, allow the identification of spatially organized biological processes, while presenting computational challenges for existing analysis approaches that ignore spatial dependencies. Here we introduce Smoother, a unified and modular framework that integrates positional information into non-spatial models via spatial priors and losses. In simulated and real datasets, we show that Smoother enables spatially aware data imputation, cell-type deconvolution, and dimensionality reduction with high accuracy.
</div>
<div id="bib_Su2022" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Su2022,
  author = {Su, Jiayu and Reynier, Jean-Baptiste and Fu, Xi and Zhong, Guojie and Jiang, Jiahao and Escalante, Rydberg Supo and Wang, Yiping and Izar, Benjamin and Knowles, David A and Rabadan, Raul},
  title = {A Unified Modular Framework to Incorporate Structural Dependency in Spatial Omics Data},
  journal = {bioRxiv},
  year = {2022},
  pages = {2022.10.25.513785},
  url = {https://www.biorxiv.org/content/10.1101/2022.10.25.513785v1.abstract}
}
</pre>
</div>
</li><li><div id="Brown2020"><em>Brown BC and Knowles DA</em> (2020), <strong>"Phenome-scale causal network discovery with bidirectional mediated Mendelian randomization"</strong>, bioRxiv. </div>
<div>[<a data-toggle="collapse" href="#abs_Brown2020">Abstract</a>][<a href="#bib_Brown2020" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/2020.06.18.160176" target="_blank">DOI</a>] [<a href="https://www.biorxiv.org/content/early/2020/06/22/2020.06.18.160176" target="_blank">URL</a>]</div>

<div id="abs_Brown2020" class="collapse">
	<b>Abstract</b>: Inference of directed biological networks from observational genomics datasets is a crucial but notoriously difficult challenge. Modern population-scale biobanks, containing simultaneous measurements of traits, biomarkers, and genetic variation, offer an unprecedented opportunity to study biological networks. Mendelian randomization (MR) has received attention as a class of methods for inferring causal effects in observational data that uses genetic variants as instrumental variables, but MR methods rely on assumptions that limit their application to complex traits at the biobank-scale. Moreover, MR estimates the total effect of one trait on another, which may be mediated by other factors. Biobanks include measurements of many potential mediators, in principle enabling the conversion of MR estimates into direct effects representing a causal network. Here, we show that this can be accomplished by a flexible two stage procedure we call bidirectional mediated Mendelian randomization (bimmer). First, bimmer estimates the effect of every trait on every other. Next, bimmer finds a parsimonious network that explains these effects using direct and mediated causal paths. We introduce novel methods for both steps and show via extensive simulations that bimmer is able to learn causal network structures even in the presence of non-causal genetic correlation. We apply bimmer to 405 phenotypes from the UK biobank and demonstrate that learning the network structure is invaluable for interpreting the results of phenome-wide MR, while lending causal support to several recent observational studies.
</div>
<div id="bib_Brown2020" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Brown2020,
  author = {Brown, Brielin C. and Knowles, David A.},
  title = {Phenome-scale causal network discovery with bidirectional mediated Mendelian randomization},
  journal = {bioRxiv},
  publisher = {Cold Spring Harbor Laboratory},
  year = {2020},
  url = {https://www.biorxiv.org/content/early/2020/06/22/2020.06.18.160176},
  doi = {10.1101/2020.06.18.160176}
}
</pre>
</div>
</li><li><div id="stirn2020variational"><em>Stirn A and Knowles DA</em> (2020), <strong>"Variational Variance: Simple and Reliable Predictive Variance Parameterization"</strong>, arXiv. </div>
<div>[<a data-toggle="collapse" href="#abs_stirn2020variational">Abstract</a>][<a href="#bib_stirn2020variational" data-toggle="collapse">BibTeX</a>] [<a href="arXiv:2006.04910v2" target="_blank">DOI</a>] [<a href="https://arxiv.org/abs/2006.04910" target="_blank">URL</a>]</div>

<div id="abs_stirn2020variational" class="collapse">
	<b>Abstract</b>: Brittle optimization has been observed to adversely impact model likelihoods for regression and VAEs when simultaneously fitting neural network mappings from a (random) variable onto the mean and variance of a dependent Gaussian variable. Previous works have bolstered optimization and improved likelihoods, but fail other basic posterior predictive checks (PPCs). Under the PPC framework, we propose critiques to test predictive mean and variance calibration and the predictive distribution's ability to generate sensible data. We find that our attractively simple solution, to treat heteroscedastic variance variationally, sufficiently regularizes variance to pass these PPCs. We consider a diverse gamut of existing and novel priors and find our methods preserve or outperform existing model likelihoods while significantly improving parameter calibration and sample quality for regression and VAEs.
</div>
<div id="bib_stirn2020variational" class="collapse">
<b>BibTeX</b>:
<pre>
@article{stirn2020variational,
  author = {Stirn, Andrew and Knowles, David A},
  title = {Variational Variance: Simple and Reliable Predictive Variance Parameterization},
  journal = {arXiv},
  year = {2020},
  url = {https://arxiv.org/abs/2006.04910},
  doi = {arXiv:2006.04910v2}
}
</pre>
</div>
</li><li><div id="Knowles2015stochastic"><em>Knowles DA</em> (2015), <strong>"Stochastic gradient variational Bayes for gamma approximating distributions"</strong>, arXiv. , pp. 1509.01631.</div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2015stochastic">Abstract</a>][<a href="#bib_Knowles2015stochastic" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/1509.01631" target="_blank">URL</a>] [<a href="https://github.com/davidaknowles/gamma_sgvb" target="_blank">Code</a>]</div>

<div id="abs_Knowles2015stochastic" class="collapse">
	<b>Abstract</b>: While stochastic variational inference is relatively well known for scaling inference in Bayesian probabilistic models, related methods also offer ways to circumnavigate the approximation of analytically intractable expectations. The key challenge in either setting is controlling the variance of gradient estimates: recent work has shown that for continuous latent variables, particularly multivariate Gaussians, this can be achieved by using the gradient of the log posterior. In this paper we apply the same idea to gamma distributed latent variables given gamma variational distributions, enabling straightforward "black box" variational inference in models where sparsity and non-negativity are appropriate. We demonstrate the method on a recently proposed gamma process model for network data, as well as a novel sparse factor analysis. We outperform generic sampling algorithms and the approach of using Gaussian variational distributions on transformed variables.
</div>
<div id="bib_Knowles2015stochastic" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2015stochastic,
  author = {Knowles, David A},
  title = {Stochastic gradient variational Bayes for gamma approximating distributions},
  journal = {arXiv},
  year = {2015},
  pages = {1509.01631},
  url = {https://arxiv.org/abs/1509.01631}
}
</pre>
</div>
</li><li><div id="salimans2014using"><em>Salimans T and Knowles DA</em> (2014), <strong>"On using control variates with stochastic approximation for variational Bayes and its connection to stochastic linear regression"</strong>, arXiv. , pp. 1401.1022.</div>
<div>[<a data-toggle="collapse" href="#abs_salimans2014using">Abstract</a>][<a href="#bib_salimans2014using" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/1401.1022" target="_blank">URL</a>]</div>

<div id="abs_salimans2014using" class="collapse">
	<b>Abstract</b>: Recently, we and several other authors have written about the possibilities of using stochastic approximation techniques for fitting variational approximations to intractable Bayesian posterior distributions. Naive implementations of stochastic approximation suffer from high variance in this setting. Several authors have therefore suggested using control variates to reduce this variance, while we have taken a different but analogous approach to reducing the variance which we call stochastic linear regression. In this note we take the former perspective and derive the ideal set of control variates for stochastic approximation variational Bayes under a certain set of assumptions. We then show that using these control variates is closely related to using the stochastic linear regression approximation technique we proposed earlier. A simple example shows that our method for constructing control variates leads to stochastic estimators with much lower variance compared to other approaches.
</div>
<div id="bib_salimans2014using" class="collapse">
<b>BibTeX</b>:
<pre>
@article{salimans2014using,
  author = {Salimans, Tim and Knowles, David A},
  title = {On using control variates with stochastic approximation for variational Bayes and its connection to stochastic linear regression},
  journal = {arXiv},
  year = {2014},
  pages = {1401.1022},
  url = {https://arxiv.org/abs/1401.1022}
}
</pre>
</div>
</li><li><div id="palla2013dependent"><em>Palla* K, Knowles* DA and Ghahramani Z</em> (2013), <strong>"A dependent partition-valued process for multitask clustering and time evolving network modeling"</strong>, arXiv. , pp. 1303.3265. *These authors contributed equally to this work.</div>
<div>[<a data-toggle="collapse" href="#abs_palla2013dependent">Abstract</a>][<a href="#bib_palla2013dependent" data-toggle="collapse">BibTeX</a>] [<a href="https://arxiv.org/abs/1303.3265" target="_blank">URL</a>]</div>

<div id="abs_palla2013dependent" class="collapse">
	<b>Abstract</b>: The fundamental aim of clustering algorithms is to partition data points. We consider tasks where the discovered partition is allowed to vary with some covariate such as space or time. One approach would be to use fragmentation-coagulation processes, but these, being Markov processes, are restricted to linear or tree structured covariate spaces. We define a partition-valued process on an arbitrary covariate space using Gaussian processes. We use the process to construct a multitask clustering model which partitions datapoints in a similar way across multiple data sources, and a time series model of network data which allows cluster assignments to vary over time. We describe sampling algorithms for inference and apply our method to defining cancer subtypes based on different types of cellular characteristics, finding regulatory modules from gene expression data from multiple human populations, and discovering time varying community structure in a social network.
</div>
<div id="bib_palla2013dependent" class="collapse">
<b>BibTeX</b>:
<pre>
@article{palla2013dependent,
  author = {Palla*, Konstantina and Knowles*, David A and Ghahramani, Zoubin},
  title = {A dependent partition-valued process for multitask clustering and time evolving network modeling},
  journal = {arXiv},
  year = {2013},
  pages = {1303.3265},
  url = {https://arxiv.org/abs/1303.3265}
}
</pre>
</div>
</li></ol>